--- db_details_importdocsql.php (original)
+++ db_details_importdocsql.php (modified)
@@ -123,14 +123,14 @@
 ?>
 
 <form method="post" action="db_details_importdocsql.php">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
     <input type="hidden" name="submit_show" value="true" />
     <input type="hidden" name="do" value="import" />
     <b>Please enter absolute path on webserver to docSQL Directory:</b>
     <br /><br />
-    &nbsp;&nbsp;&nbsp;&nbsp;<input type="text" name="docpath" size="50" value="<?php echo htmlspecialchars($DOCUMENT_ROOT); ?>" />
+    &nbsp;&nbsp;&nbsp;&nbsp;<input type="text" name="docpath" size="50" value="<?php echo  htmlspecialchars($DOCUMENT_ROOT) ?>" />
     &nbsp;<input type="submit" value="Import files" />
 </form>
 