--- header.inc.php (original)
+++ header.inc.php (modified)
@@ -48,11 +48,11 @@
 ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $GLOBALS['available_languages'][$GLOBALS['lang']][2]; ?>" lang="<?php echo $GLOBALS['available_languages'][$GLOBALS['lang']][2]; ?>" dir="<?php echo $GLOBALS['text_dir']; ?>">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo  $GLOBALS['available_languages'][$GLOBALS['lang']][2] ?>" lang="<?php echo  $GLOBALS['available_languages'][$GLOBALS['lang']][2] ?>" dir="<?php echo  $GLOBALS['text_dir'] ?>">
 
 <head>
 <title>phpMyAdmin</title>
-<meta http-equiv="Content-Type" content="text/html; charset=<?php echo $GLOBALS['charset']; ?>" />
+<meta http-equiv="Content-Type" content="text/html; charset=<?php echo  $GLOBALS['charset'] ?>" />
 <?php
 if (!empty($GLOBALS['cfg']['PmaAbsoluteUri'])) {
     echo '<base href="' . $GLOBALS['cfg']['PmaAbsoluteUri'] . '" />' . "\n";
@@ -61,8 +61,8 @@
 <style type="text/css">
 <!--
 body {
-    font-family: <?php echo $right_font_family; ?>;
-    font-size: <?php echo $font_size; ?>;
+    font-family: <?php echo  $right_font_family ?>;
+    font-size: <?php echo  $font_size ?>;
     color: #000000;
 <?php
 if ($GLOBALS['cfg']['RightBgImage'] == '') {
@@ -72,31 +72,31 @@
     echo '    background-image: url(\'' . $GLOBALS['cfg']['RightBgImage'] . '\');' . "\n";
 } // end if... else...
 ?>
-    background-color: <?php echo $GLOBALS['cfg']['RightBgColor'] . "\n"; ?>
-}
-pre, tt         {font-size: <?php echo $font_size; ?>}
-th              {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; font-weight: bold; color: #000000; background-color: <?php echo $GLOBALS['cfg']['ThBgcolor']; ?>}
-td              {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>}
-form            {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>}
-input           {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>}
-input.textfield {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000; background-color: #FFFFFF}
-select          {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000; background-color: #FFFFFF}
-textarea        {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000; background-color: #FFFFFF}
-h1              {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold}
-a:link          {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; text-decoration: none; color: #0000FF}
-a:visited       {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; text-decoration: none; color: #0000FF}
-a:hover         {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; text-decoration: underline; color: #FF0000}
-a.nav:link      {font-family: <?php echo $right_font_family; ?>; color: #000000}
-a.nav:visited   {font-family: <?php echo $right_font_family; ?>; color: #000000}
-a.nav:hover     {font-family: <?php echo $right_font_family; ?>; color: #FF0000}
-a.h1:link       {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold; color: #000000}
-a.h1:visited    {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold; color: #000000}
-a.h1:hover      {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold; color: #FF0000}
-a.drop:link     {font-family: <?php echo $right_font_family; ?>; color: #ff0000}
-a.drop:visited  {font-family: <?php echo $right_font_family; ?>; color: #ff0000}
-a.drop:hover    {font-family: <?php echo $right_font_family; ?>; color: #ffffff; background-color:#ff0000; text-decoration: none}
-.nav            {font-family: <?php echo $right_font_family; ?>; color: #000000}
-.warning        {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; font-weight: bold; color: #FF0000}
+    background-color: <?php echo  $GLOBALS['cfg']['RightBgColor'] . "\n" ?>
+}
+pre, tt         {font-size: <?php echo  $font_size ?>}
+th              {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; font-weight: bold; color: #000000; background-color: <?php echo  $GLOBALS['cfg']['ThBgcolor'] ?>}
+td              {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>}
+form            {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>}
+input           {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>}
+input.textfield {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000; background-color: #FFFFFF}
+select          {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000; background-color: #FFFFFF}
+textarea        {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000; background-color: #FFFFFF}
+h1              {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold}
+a:link          {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; text-decoration: none; color: #0000FF}
+a:visited       {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; text-decoration: none; color: #0000FF}
+a:hover         {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; text-decoration: underline; color: #FF0000}
+a.nav:link      {font-family: <?php echo  $right_font_family ?>; color: #000000}
+a.nav:visited   {font-family: <?php echo  $right_font_family ?>; color: #000000}
+a.nav:hover     {font-family: <?php echo  $right_font_family ?>; color: #FF0000}
+a.h1:link       {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold; color: #000000}
+a.h1:visited    {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold; color: #000000}
+a.h1:hover      {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold; color: #FF0000}
+a.drop:link     {font-family: <?php echo  $right_font_family ?>; color: #ff0000}
+a.drop:visited  {font-family: <?php echo  $right_font_family ?>; color: #ff0000}
+a.drop:hover    {font-family: <?php echo  $right_font_family ?>; color: #ffffff; background-color:#ff0000; text-decoration: none}
+.nav            {font-family: <?php echo  $right_font_family ?>; color: #000000}
+.warning        {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; font-weight: bold; color: #FF0000}
 td.topline      {font-size: 1px}
 td.tab          {
     border-top: 1px solid #999;
@@ -115,7 +115,7 @@
 
 .print{font-family:arial;font-size:8pt;}
 
-.syntax {font-family: sans-serif; font-size: <?php echo $font_smaller; ?>;}
+.syntax {font-family: sans-serif; font-size: <?php echo  $font_smaller ?>;}
 .syntax_comment            {}
 .syntax_digit              {}
 .syntax_digit_hex          {}
@@ -131,9 +131,7 @@
 .syntax_alpha_variable     {}
 .syntax_quote              {}
 .syntax_quote_backtick     {}
-<?php
-echo PMA_SQP_buildCssData();
-?>
+<?php echo  PMA_SQP_buildCssData() ?>
 //-->
 </style>
 
@@ -156,7 +154,7 @@
 // Updates the title of the frameset if possible (ns4 does not allow this)
 if (typeof(parent.document) != 'undefined' && typeof(parent.document) != 'unknown'
     && typeof(parent.document.title) == 'string') {
-    parent.document.title = '<?php echo $title; ?>';
+    parent.document.title = '<?php echo  $title ?>';
 }
 <?php
 // Add some javascript instructions if required
@@ -164,9 +162,9 @@
     echo "\n";
     ?>
 // js form validation stuff
-var errorMsg0   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strFormEmpty']); ?>';
-var errorMsg1   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strNotNumber']); ?>';
-var errorMsg2   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strNotValidNumber']); ?>';
+var errorMsg0   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strFormEmpty']) ?>';
+var errorMsg1   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strNotNumber']) ?>';
+var errorMsg2   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strNotValidNumber']) ?>';
 var noDropDbMsg = '<?php echo((!$GLOBALS['cfg']['AllowUserDropDatabase']) ? str_replace('\'', '\\\'', $GLOBALS['strNoDropDatabases']) : ''); ?>';
 var confirmMsg  = '<?php echo(($GLOBALS['cfg']['Confirm']) ? str_replace('\'', '\\\'', $GLOBALS['strDoYouReally']) : ''); ?>';
 //-->
@@ -177,10 +175,10 @@
     echo "\n";
     ?>
 // js form validation stuff
-var jsHostEmpty       = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strHostEmpty']); ?>';
-var jsUserEmpty       = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strUserEmpty']); ?>';
-var jsPasswordEmpty   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strPasswordEmpty']); ?>';
-var jsPasswordNotSame = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strPasswordNotSame']); ?>';
+var jsHostEmpty       = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strHostEmpty']) ?>';
+var jsUserEmpty       = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strUserEmpty']) ?>';
+var jsPasswordEmpty   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strPasswordEmpty']) ?>';
+var jsPasswordNotSame = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strPasswordNotSame']) ?>';
 //-->
 </script>
 <script src="libraries/user_details.js" type="text/javascript" language="javascript"></script>
@@ -189,9 +187,9 @@
     echo "\n";
     ?>
 // js index validation stuff
-var errorMsg0   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strFormEmpty']); ?>';
-var errorMsg1   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strNotNumber']); ?>';
-var errorMsg2   = '<?php echo str_replace('\'', '\\\'', $GLOBALS['strNotValidNumber']); ?>';
+var errorMsg0   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strFormEmpty']) ?>';
+var errorMsg1   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strNotNumber']) ?>';
+var errorMsg2   = '<?php echo  str_replace('\'', '\\\'', $GLOBALS['strNotValidNumber']) ?>';
 //-->
 </script>
 <script src="libraries/indexes.js" type="text/javascript" language="javascript"></script>
@@ -222,7 +220,7 @@
     $bkg_img = '';
 }
 ?>
-<body bgcolor="<?php echo $GLOBALS['cfg']['RightBgColor'] . '"' . $bkg_img; ?>>
+<body bgcolor="<?php echo  $GLOBALS['cfg']['RightBgColor'] . '"' . $bkg_img ?>>
 <?php
 if (isset($GLOBALS['db'])) {
     $header_url_qry = '?lang=' . urlencode($GLOBALS['lang'])