--- read_dump.php (original)
+++ read_dump.php (modified)
@@ -131,7 +131,7 @@
     } // end for
 
     // add any rest to the returned array
-    if (!empty($sql) && ereg('[^[:space:]]+', $sql)) {
+    if (!empty($sql) && preg_match('[^[:space:]]+', $sql)) {
         $ret[] = $sql;
     }
 
@@ -160,7 +160,7 @@
         $tmp_file     .= '/' . basename($filename);
 
         // User might have trailing slash in php.ini...
-        return (ereg_replace('/+', '/', $tmp_file) == $filename);
+        return (preg_replace('/+', '/', $tmp_file) == $filename);
     } // end of the 'is_uploaded_file()' emulated function
 } // end if
 
@@ -360,7 +360,7 @@
 
 // Drop database is not allowed -> ensure the query can be run
 if (!$cfg['AllowUserDropDatabase']
-    && eregi('DROP[[:space:]]+(IF EXISTS[[:space:]]+)?DATABASE ', $sql_query)) {
+    && preg_match('DROP[[:space:]]+(IF EXISTS[[:space:]]+)?DATABASE ', $sql_query)) {
     // Checks if the user is a Superuser
     // TODO: set a global variable with this information
     // loic1: optimized query
@@ -408,7 +408,7 @@
             } else {
                 $sql_query = $pieces[0];
             }
-            if (eregi('^(DROP|CREATE)[[:space:]]+(IF EXISTS[[:space:]]+)?(TABLE|DATABASE)[[:space:]]+(.+)', $sql_query)) {
+            if (preg_match('^(DROP|CREATE)[[:space:]]+(IF EXISTS[[:space:]]+)?(TABLE|DATABASE)[[:space:]]+(.+)', $sql_query)) {
                 $reload = 1;
             }
             include('./sql.php');
@@ -425,7 +425,7 @@
                     $my_die = $a_sql_query;
                     break;
                 }
-                if (!isset($reload) && eregi('^(DROP|CREATE)[[:space:]]+(IF EXISTS[[:space:]]+)?(TABLE|DATABASE)[[:space:]]+(.+)', $a_sql_query)) {
+                if (!isset($reload) && preg_match('^(DROP|CREATE)[[:space:]]+(IF EXISTS[[:space:]]+)?(TABLE|DATABASE)[[:space:]]+(.+)', $a_sql_query)) {
                     $reload = 1;
                 }
             } // end for
@@ -461,7 +461,7 @@
         $goto     = 'db_details.php';
     } else {
         $is_table = @PMA_mysql_query('SHOW TABLES LIKE \'' . PMA_sqlAddslashes($table, TRUE) . '\'');
-        if (!($is_table && @mysql_numrows($is_table))) {
+        if (!($is_table && @mysqli_numrows($is_table))) {
             $goto = 'db_details.php';
             unset($table);
         }