--- tbl_properties_structure.php (original)
+++ tbl_properties_structure.php (modified)
@@ -40,12 +40,12 @@
         $pk_array[$row['Column_name']] = 1;
     }
 } // end while
-mysql_free_result($result);
+mysqli_free_result($result);
 
 // 3. Get fields
 $local_query = 'SHOW FIELDS FROM ' . PMA_backquote($table);
 $fields_rs   = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, '', $err_url_0);
-$fields_cnt  = mysql_num_rows($fields_rs);
+$fields_cnt  = mysqli_num_rows($fields_rs);
 
 
 
@@ -57,21 +57,21 @@
 <!-- TABLE INFORMATIONS -->
 
 <form action="tbl_properties_structure.php">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-    <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
-
-<table border="<?php echo $cfg['Border']; ?>">
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+    <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
+
+<table border="<?php echo  $cfg['Border'] ?>">
 <tr>
     <td></td>
-    <th>&nbsp;<?php echo $strField; ?>&nbsp;</th>
-    <th><?php echo $strType; ?></th>
-    <th><?php echo $strAttr; ?></th>
-    <th><?php echo $strNull; ?></th>
-    <th><?php echo $strDefault; ?></th>
-    <th><?php echo $strExtra; ?></th>
-    <th colspan="<?php echo((PMA_MYSQL_INT_VERSION >= 32323) ? '6' : '5'); ?>"><?php echo $strAction; ?></th>
+    <th>&nbsp;<?php echo  $strField ?>&nbsp;</th>
+    <th><?php echo  $strType ?></th>
+    <th><?php echo  $strAttr ?></th>
+    <th><?php echo  $strNull ?></th>
+    <th><?php echo  $strDefault ?></th>
+    <th><?php echo  $strExtra ?></th>
+    <th colspan="<?php echo((PMA_MYSQL_INT_VERSION >= 32323) ? '6' : '5'); ?>"><?php echo  $strAction ?></th>
 </tr>
 
 <?php
@@ -86,23 +86,23 @@
     $type             = $row['Type'];
     // reformat mysql query output - staybyte - 9. June 2001
     // loic1: set or enum types: slashes single quotes inside options
-    if (eregi('^(set|enum)\((.+)\)$', $type, $tmp)) {
-        $tmp[2]       = substr(ereg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
+    if (preg_match('^(set|enum)\((.+)\)$', $type, $tmp)) {
+        $tmp[2]       = substr(preg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
         $type         = $tmp[1] . '(' . str_replace(',', ', ', $tmp[2]) . ')';
         $type_nowrap  = '';
     } else {
         $type_nowrap  = ' nowrap="nowrap"';
     }
-    $type             = eregi_replace('BINARY', '', $type);
-    $type             = eregi_replace('ZEROFILL', '', $type);
-    $type             = eregi_replace('UNSIGNED', '', $type);
+    $type             = preg_replace('BINARY', '', $type);
+    $type             = preg_replace('ZEROFILL', '', $type);
+    $type             = preg_replace('UNSIGNED', '', $type);
     if (empty($type)) {
         $type         = '&nbsp;';
     }
 
-    $binary           = eregi('BINARY', $row['Type'], $test);
-    $unsigned         = eregi('UNSIGNED', $row['Type'], $test);
-    $zerofill         = eregi('ZEROFILL', $row['Type'], $test);
+    $binary           = preg_match('BINARY', $row['Type'], $test);
+    $unsigned         = preg_match('UNSIGNED', $row['Type'], $test);
+    $zerofill         = preg_match('ZEROFILL', $row['Type'], $test);
     $strAttribute     = '&nbsp;';
     if ($binary) {
         $strAttribute = 'BINARY';
@@ -130,28 +130,28 @@
 
     ?>
 <tr>
-    <td align="center" bgcolor="<?php echo $bgcolor; ?>">
-        <input type="checkbox" name="selected_fld[]" value="<?php echo $field_encoded; ?>" id="checkbox_row_<?php echo $i; ?>" />
-    </td>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap">&nbsp;<label for="checkbox_row_<?php echo $i; ?>"><?php echo $field_name; ?></label>&nbsp;</td>
-    <td bgcolor="<?php echo $bgcolor; ?>"<?php echo $type_nowrap; ?>><?php echo $type; ?><bdo dir="ltr"></bdo></td>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $strAttribute; ?></td>
-    <td bgcolor="<?php echo $bgcolor; ?>"><?php echo (($row['Null'] == '') ? $strNo : $strYes); ?>&nbsp;</td>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php if (isset($row['Default'])) echo $row['Default']; ?>&nbsp;</td>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $row['Extra']; ?>&nbsp;</td>
-    <td bgcolor="<?php echo $bgcolor; ?>">
-        <a href="tbl_alter.php?<?php echo $url_query; ?>&amp;field=<?php echo $field_encoded; ?>">
-            <?php echo $strChange; ?></a>
-    </td>
-    <td bgcolor="<?php echo $bgcolor; ?>">
+    <td align="center" bgcolor="<?php echo  $bgcolor ?>">
+        <input type="checkbox" name="selected_fld[]" value="<?php echo  $field_encoded ?>" id="checkbox_row_<?php echo  $i ?>" />
+    </td>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap">&nbsp;<label for="checkbox_row_<?php echo  $i ?>"><?php echo  $field_name ?></label>&nbsp;</td>
+    <td bgcolor="<?php echo  $bgcolor ?>"<?php echo  $type_nowrap ?>><?php echo  $type ?><bdo dir="ltr"></bdo></td>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $strAttribute ?></td>
+    <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  (($row['Null'] == '') ? $strNo : $strYes) ?>&nbsp;</td>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php if (isset($row['Default'])) echo $row['Default']; ?>&nbsp;</td>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $row['Extra'] ?>&nbsp;</td>
+    <td bgcolor="<?php echo  $bgcolor ?>">
+        <a href="tbl_alter.php?<?php echo  $url_query ?>&amp;field=<?php echo  $field_encoded ?>">
+            <?php echo  $strChange ?></a>
+    </td>
+    <td bgcolor="<?php echo  $bgcolor ?>">
         <?php
         // loic1: Drop field only if there is more than one field in the table
         if ($fields_cnt > 1) {
             echo "\n";
             ?>
-        <a href="sql.php?<?php echo $url_query; ?>&amp;sql_query=<?php echo urlencode('ALTER TABLE ' . PMA_backquote($table) . ' DROP ' . PMA_backquote($row['Field'])); ?>&amp;zero_rows=<?php echo urlencode(sprintf($strFieldHasBeenDropped, htmlspecialchars($row['Field']))); ?>"
-            onclick="return confirmLink(this, 'ALTER TABLE <?php echo PMA_jsFormat($table); ?> DROP <?php echo PMA_jsFormat($row['Field']); ?>')">
-            <?php echo $strDrop; ?></a>
+        <a href="sql.php?<?php echo  $url_query ?>&amp;sql_query=<?php echo  urlencode('ALTER TABLE ' . PMA_backquote($table) . ' DROP ' . PMA_backquote($row['Field'])) ?>&amp;zero_rows=<?php echo  urlencode(sprintf($strFieldHasBeenDropped, htmlspecialchars($row['Field']))) ?>"
+            onclick="return confirmLink(this, 'ALTER TABLE <?php echo  PMA_jsFormat($table) ?> DROP <?php echo  PMA_jsFormat($row['Field']) ?>')">
+            <?php echo  $strDrop ?></a>
             <?php
         } else {
             echo "\n" . '        ' . $strDrop;
@@ -159,44 +159,44 @@
         echo "\n";
         ?>
     </td>
-    <td bgcolor="<?php echo $bgcolor; ?>">
+    <td bgcolor="<?php echo  $bgcolor ?>">
         <?php
         if ($type == 'text' || $type == 'blob') {
             echo $strPrimary . "\n";
         } else {
             echo "\n";
             ?>
-        <a href="sql.php?<?php echo $url_query; ?>&amp;sql_query=<?php echo urlencode('ALTER TABLE ' . PMA_backquote($table) . ' DROP PRIMARY KEY, ADD PRIMARY KEY(' . $primary . PMA_backquote($row['Field']) . ')'); ?>&amp;zero_rows=<?php echo urlencode(sprintf($strAPrimaryKey, htmlspecialchars($row['Field']))); ?>"
-            onclick="return confirmLink(this, 'ALTER TABLE <?php echo PMA_jsFormat($table); ?> DROP PRIMARY KEY, ADD PRIMARY KEY(<?php echo PMA_jsFormat($row['Field']); ?>)')">
-            <?php echo $strPrimary; ?></a>
+        <a href="sql.php?<?php echo  $url_query ?>&amp;sql_query=<?php echo  urlencode('ALTER TABLE ' . PMA_backquote($table) . ' DROP PRIMARY KEY, ADD PRIMARY KEY(' . $primary . PMA_backquote($row['Field']) . ')') ?>&amp;zero_rows=<?php echo  urlencode(sprintf($strAPrimaryKey, htmlspecialchars($row['Field']))) ?>"
+            onclick="return confirmLink(this, 'ALTER TABLE <?php echo  PMA_jsFormat($table) ?> DROP PRIMARY KEY, ADD PRIMARY KEY(<?php echo  PMA_jsFormat($row['Field']) ?>)')">
+            <?php echo  $strPrimary ?></a>
             <?php
         }
         echo "\n";
         ?>
     </td>
-    <td bgcolor="<?php echo $bgcolor; ?>">
+    <td bgcolor="<?php echo  $bgcolor ?>">
         <?php
         if ($type == 'text' || $type == 'blob') {
             echo $strIndex . "\n";
         } else {
             echo "\n";
             ?>
-        <a href="sql.php?<?php echo $url_query; ?>&amp;sql_query=<?php echo urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD INDEX(' . PMA_backquote($row['Field']) . ')'); ?>&amp;zero_rows=<?php echo urlencode(sprintf($strAnIndex ,htmlspecialchars($row['Field']))); ?>">
-            <?php echo $strIndex; ?></a>
+        <a href="sql.php?<?php echo  $url_query ?>&amp;sql_query=<?php echo  urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD INDEX(' . PMA_backquote($row['Field']) . ')') ?>&amp;zero_rows=<?php echo  urlencode(sprintf($strAnIndex ,htmlspecialchars($row['Field']))) ?>">
+            <?php echo  $strIndex ?></a>
             <?php
         }
         echo "\n";
         ?>
     </td>
-    <td bgcolor="<?php echo $bgcolor; ?>">
+    <td bgcolor="<?php echo  $bgcolor ?>">
         <?php
         if ($type == 'text' || $type == 'blob') {
             echo $strUnique . "\n";
         } else {
             echo "\n";
             ?>
-        <a href="sql.php?<?php echo $url_query; ?>&amp;sql_query=<?php echo urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD UNIQUE(' . PMA_backquote($row['Field']) . ')'); ?>&amp;zero_rows=<?php echo urlencode(sprintf($strAnIndex , htmlspecialchars($row['Field']))); ?>">
-            <?php echo $strUnique; ?></a>
+        <a href="sql.php?<?php echo  $url_query ?>&amp;sql_query=<?php echo  urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD UNIQUE(' . PMA_backquote($row['Field']) . ')') ?>&amp;zero_rows=<?php echo  urlencode(sprintf($strAnIndex , htmlspecialchars($row['Field']))) ?>">
+            <?php echo  $strUnique ?></a>
             <?php
         }
         echo "\n";
@@ -208,16 +208,16 @@
             && ($type == 'text' || strpos(' ' . $type, 'varchar'))) {
             echo "\n";
             ?>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap">
-        <a href="sql.php?<?php echo $url_query; ?>&amp;sql_query=<?php echo urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD FULLTEXT(' . PMA_backquote($row['Field']) . ')'); ?>&amp;zero_rows=<?php echo urlencode(sprintf($strAnIndex , htmlspecialchars($row['Field']))); ?>">
-            <?php echo $strIdxFulltext; ?></a>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap">
+        <a href="sql.php?<?php echo  $url_query ?>&amp;sql_query=<?php echo  urlencode('ALTER TABLE ' . PMA_backquote($table) . ' ADD FULLTEXT(' . PMA_backquote($row['Field']) . ')') ?>&amp;zero_rows=<?php echo  urlencode(sprintf($strAnIndex , htmlspecialchars($row['Field']))) ?>">
+            <?php echo  $strIdxFulltext ?></a>
     </td>
             <?php
         } else {
             echo "\n";
             ?>
-    <td bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap">
-        <?php echo $strIdxFulltext . "\n"; ?>
+    <td bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap">
+        <?php echo  $strIdxFulltext . "\n" ?>
     </td>
             <?php
         } // end if... else...
@@ -233,15 +233,15 @@
 
 <tr>
     <td colspan="<?php echo((PMA_MYSQL_INT_VERSION >= 32323) ? '13' : '12'); ?>">
-        <img src="./images/arrow_<?php echo $text_dir; ?>.gif" border="0" width="38" height="22" alt="<?php echo $strWithChecked; ?>" />
-        <i><?php echo $strWithChecked; ?></i>&nbsp;&nbsp;
-        <input type="submit" name="submit_mult" value="<?php echo $strChange; ?>" />
+        <img src="./images/arrow_<?php echo  $text_dir ?>.gif" border="0" width="38" height="22" alt="<?php echo  $strWithChecked ?>" />
+        <i><?php echo  $strWithChecked ?></i>&nbsp;&nbsp;
+        <input type="submit" name="submit_mult" value="<?php echo  $strChange ?>" />
 <?php
 // Drop button if there is at least two fields
 if ($fields_cnt > 1) {
     ?>
-        &nbsp;<i><?php echo $strOr; ?></i>&nbsp;
-        <input type="submit" name="submit_mult" value="<?php echo $strDrop; ?>" />
+        &nbsp;<i><?php echo  $strOr ?></i>&nbsp;
+        <input type="submit" name="submit_mult" value="<?php echo  $strDrop ?>" />
     <?php
 }
 echo "\n";
@@ -294,7 +294,7 @@
 if ($cfg['ShowStats']) {
     $nonisam     = FALSE;
     $is_innodb = (isset($showtable['Type']) && $showtable['Type'] == 'InnoDB');
-    if (isset($showtable['Type']) && !eregi('ISAM|HEAP', $showtable['Type'])) {
+    if (isset($showtable['Type']) && !preg_match('ISAM|HEAP', $showtable['Type'])) {
         $nonisam = TRUE;
     }
     if (PMA_MYSQL_INT_VERSION >= 32303 && ($nonisam == FALSE || $is_innodb)) {
@@ -324,26 +324,26 @@
     <!-- Space usage -->
     <td width="20">&nbsp;</td>
     <td valign="top">
-        <?php echo $strSpaceUsage . '&nbsp;:' . "\n"; ?>
+        <?php echo  $strSpaceUsage . '&nbsp;:' . "\n" ?>
         <a name="showusage"></a>
-        <table border="<?php echo $cfg['Border']; ?>">
-        <tr>
-            <th><?php echo $strType; ?></th>
-            <th colspan="2" align="center"><?php echo $strUsage; ?></th>
-        </tr>
-        <tr>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" style="padding-right: 10px"><?php echo $strData; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" align="right" nowrap="nowrap"><?php echo $data_size; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>"><?php echo $data_unit; ?></td>
+        <table border="<?php echo  $cfg['Border'] ?>">
+        <tr>
+            <th><?php echo  $strType ?></th>
+            <th colspan="2" align="center"><?php echo  $strUsage ?></th>
+        </tr>
+        <tr>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" style="padding-right: 10px"><?php echo  $strData ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" align="right" nowrap="nowrap"><?php echo  $data_size ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>"><?php echo  $data_unit ?></td>
         </tr>
         <?php
         if (isset($index_size)) {
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" style="padding-right: 10px"><?php echo $strIndex; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" align="right" nowrap="nowrap"><?php echo $index_size; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>"><?php echo $index_unit; ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" style="padding-right: 10px"><?php echo  $strIndex ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" align="right" nowrap="nowrap"><?php echo  $index_size ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>"><?php echo  $index_unit ?></td>
         </tr>
             <?php
         }
@@ -351,14 +351,14 @@
             echo "\n";
             ?>
         <tr style="color: #bb0000">
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" style="padding-right: 10px"><?php echo $strOverhead; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>" align="right" nowrap="nowrap"><?php echo $free_size; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorTwo']; ?>"><?php echo $free_unit; ?></td>
-        </tr>
-        <tr>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>" style="padding-right: 10px"><?php echo $strEffective; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>" align="right" nowrap="nowrap"><?php echo $effect_size; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>"><?php echo $effect_unit; ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" style="padding-right: 10px"><?php echo  $strOverhead ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>" align="right" nowrap="nowrap"><?php echo  $free_size ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorTwo'] ?>"><?php echo  $free_unit ?></td>
+        </tr>
+        <tr>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>" style="padding-right: 10px"><?php echo  $strEffective ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>" align="right" nowrap="nowrap"><?php echo  $effect_size ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>"><?php echo  $effect_unit ?></td>
         </tr>
             <?php
         }
@@ -366,9 +366,9 @@
             echo "\n";
         ?>
         <tr>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>" style="padding-right: 10px"><?php echo $strTotalUC; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>" align="right" nowrap="nowrap"><?php echo $tot_size; ?></td>
-            <td bgcolor="<?php echo $cfg['BgcolorOne']; ?>"><?php echo $tot_unit; ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>" style="padding-right: 10px"><?php echo  $strTotalUC ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>" align="right" nowrap="nowrap"><?php echo  $tot_size ?></td>
+            <td bgcolor="<?php echo  $cfg['BgcolorOne'] ?>"><?php echo  $tot_unit ?></td>
         </tr>
             <?php
         }
@@ -378,7 +378,7 @@
             ?>
         <tr>
             <td colspan="3" align="center">
-                [<a href="sql.php?<?php echo $url_query; ?>&amp;pos=0&amp;sql_query=<?php echo urlencode('OPTIMIZE TABLE ' . PMA_backquote($table)); ?>"><?php echo $strOptimizeTable; ?></a>]
+                [<a href="sql.php?<?php echo  $url_query ?>&amp;pos=0&amp;sql_query=<?php echo  urlencode('OPTIMIZE TABLE ' . PMA_backquote($table)) ?>"><?php echo  $strOptimizeTable ?></a>]
             </td>
         </tr>
             <?php
@@ -391,11 +391,11 @@
     <!-- Rows Statistic -->
     <td width="20">&nbsp;</td>
     <td valign="top">
-        <?php echo $strRowsStatistic . '&nbsp;:' . "\n"; ?>
-        <table border="<?php echo $cfg['Border']; ?>">
-        <tr>
-            <th><?php echo $strStatement; ?></th>
-            <th align="center"><?php echo $strValue; ?></th>
+        <?php echo  $strRowsStatistic . '&nbsp;:' . "\n" ?>
+        <table border="<?php echo  $cfg['Border'] ?>">
+        <tr>
+            <th><?php echo  $strStatement ?></th>
+            <th align="center"><?php echo  $strValue ?></th>
         </tr>
         <?php
         $i = 0;
@@ -404,8 +404,8 @@
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $bgcolor; ?>"><?php echo $strFormat; ?></td>
-            <td bgcolor="<?php echo $bgcolor; ?>" align="<?php echo $cell_align_left; ?>" nowrap="nowrap">
+            <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  $strFormat ?></td>
+            <td bgcolor="<?php echo  $bgcolor ?>" align="<?php echo  $cell_align_left ?>" nowrap="nowrap">
             <?php
             echo '                ';
             if ($showtable['Row_format'] == 'Fixed') {
@@ -428,9 +428,9 @@
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $bgcolor; ?>"><?php echo $strRows; ?></td>
-            <td bgcolor="<?php echo $bgcolor; ?>" align="right" nowrap="nowrap">
-                <?php echo number_format($showtable['Rows'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+            <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  $strRows ?></td>
+            <td bgcolor="<?php echo  $bgcolor ?>" align="right" nowrap="nowrap">
+                <?php echo  number_format($showtable['Rows'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
             <?php
@@ -440,9 +440,9 @@
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $bgcolor; ?>"><?php echo $strRowLength; ?>&nbsp;&oslash;</td>
-            <td bgcolor="<?php echo $bgcolor; ?>" align="right" nowrap="nowrap">
-                <?php echo number_format($showtable['Avg_row_length'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+            <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  $strRowLength ?>&nbsp;&oslash;</td>
+            <td bgcolor="<?php echo  $bgcolor ?>" align="right" nowrap="nowrap">
+                <?php echo  number_format($showtable['Avg_row_length'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
             <?php
@@ -452,9 +452,9 @@
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $bgcolor; ?>"><?php echo $strRowSize; ?>&nbsp;&oslash;</td>
-            <td bgcolor="<?php echo $bgcolor; ?>" align="right" nowrap="nowrap">
-                <?php echo $avg_size . ' ' . $avg_unit . "\n"; ?>
+            <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  $strRowSize ?>&nbsp;&oslash;</td>
+            <td bgcolor="<?php echo  $bgcolor ?>" align="right" nowrap="nowrap">
+                <?php echo  $avg_size . ' ' . $avg_unit . "\n" ?>
             </td>
         </tr>
             <?php
@@ -464,9 +464,9 @@
             echo "\n";
             ?>
         <tr>
-            <td bgcolor="<?php echo $bgcolor; ?>"><?php echo $strNext; ?>&nbsp;Autoindex</td>
-            <td bgcolor="<?php echo $bgcolor; ?>" align="right" nowrap="nowrap">
-                <?php echo number_format($showtable['Auto_increment'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+            <td bgcolor="<?php echo  $bgcolor ?>"><?php echo  $strNext ?>&nbsp;Autoindex</td>
+            <td bgcolor="<?php echo  $bgcolor ?>" align="right" nowrap="nowrap">
+                <?php echo  number_format($showtable['Auto_increment'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
             <?php
@@ -496,22 +496,22 @@
 
     <!-- Printable view of the table -->
     <li>
-        <div style="margin-bottom: 10px"><a href="tbl_printview.php?<?php echo $url_query; ?>"><?php echo $strPrintView; ?></a></div>
+        <div style="margin-bottom: 10px"><a href="tbl_printview.php?<?php echo  $url_query ?>"><?php echo  $strPrintView ?></a></div>
     </li>
 
     <!-- Add some new fields -->
     <li>
         <form method="post" action="tbl_addfield.php"
             onsubmit="return checkFormElementInRange(this, 'num_fields', 1)">
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-            <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
-            <?php echo $strAddNewField; ?>&nbsp;:
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+            <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
+            <?php echo  $strAddNewField ?>&nbsp;:
             <input type="text" name="num_fields" size="2" maxlength="2" value="1" class="textfield" style="vertical-align: middle" onfocus="this.select()" />
             <select name="after_field" style="vertical-align: middle">
-                <option value="--end--"><?php echo $strAtEndOfTable; ?></option>
-                <option value="--first--"><?php echo $strAtBeginningOfTable; ?></option>
+                <option value="--end--"><?php echo  $strAtEndOfTable ?></option>
+                <option value="--first--"><?php echo  $strAtBeginningOfTable ?></option>
 <?php
 reset($aryFields);
 while (list($junk, $fieldname) = each($aryFields)) {
@@ -520,7 +520,7 @@
 unset($aryFields);
 ?>
             </select>
-            <input type="submit" value="<?php echo $strGo; ?>" style="vertical-align: middle" />
+            <input type="submit" value="<?php echo  $strGo ?>" style="vertical-align: middle" />
         </form>
     </li>
 
@@ -530,7 +530,7 @@
     <!-- Work on Relations -->
     <li>
         <div style="margin-bottom: 10px">
-            <a href="tbl_relation.php?<?php echo $url_query; ?>"><?php echo $strRelationView; ?></a>
+            <a href="tbl_relation.php?<?php echo  $url_query ?>"><?php echo  $strRelationView ?></a>
         </div>
     </li>
     <?php
@@ -541,9 +541,9 @@
     <!-- Let MySQL propose the optimal structure -->
     <li>
         <div style="margin-bottom: 10px">
-        <a href="sql.php?<?php echo $url_query; ?>&amp;session_max_rows=all&amp;sql_query=<?php echo urlencode('SELECT * FROM ' . PMA_backquote($table) . ' PROCEDURE ANALYSE()'); ?>">
-            <?php echo $strStructPropose; ?></a>
-        <?php echo PMA_showMySQLDocu('Extending_MySQL', 'procedure_analyse') . "\n"; ?>
+        <a href="sql.php?<?php echo  $url_query ?>&amp;session_max_rows=all&amp;sql_query=<?php echo  urlencode('SELECT * FROM ' . PMA_backquote($table) . ' PROCEDURE ANALYSE()') ?>">
+            <?php echo  $strStructPropose ?></a>
+        <?php echo  PMA_showMySQLDocu('Extending_MySQL', 'procedure_analyse') . "\n" ?>
         </div>
     </li>
 