--- user_password.php (original)
+++ user_password.php (modified)
@@ -56,7 +56,7 @@
 
         $err_url          = 'user_password.php?' . $common_url_query;
 
-        $sql_query        = 'SET password = ' . (($pma_pw == '') ? '\'\'' : 'PASSWORD(\'' . ereg_replace('.', '*', $pma_pw) . '\')');
+        $sql_query        = 'SET password = ' . (($pma_pw == '') ? '\'\'' : 'PASSWORD(\'' . preg_replace('.', '*', $pma_pw) . '\')');
         $local_query      = 'SET password = ' . (($pma_pw == '') ? '\'\'' : 'PASSWORD(\'' . PMA_sqlAddslashes($pma_pw) . '\')');
         $result           = @PMA_mysql_query($local_query) or PMA_mysqlDie('', '', FALSE, $err_url);
 
@@ -76,8 +76,8 @@
         $show_query = 'y';
         PMA_showMessage(get_magic_quotes_gpc() ? addslashes($strUpdateProfileMessage) : $strUpdateProfileMessage);
         ?>
-        <a href="index.php?<?php echo $common_url_query . $http_logout; ?>" target="_parent">
-            <b><?php echo $strBack; ?></b></a>
+        <a href="index.php?<?php echo  $common_url_query . $http_logout ?>" target="_parent">
+            <b><?php echo  $strBack ?></b></a>
         <?php
         exit();
     } // end if
@@ -107,25 +107,25 @@
 // Displays the form
 ?>
 <form method="post" action="./user_password.php" name="chgPassword" onsubmit="return checkPassword(this)">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
     <table border="0">
     <tr>
         <td colspan="2">
             <input type="radio" name="nopass" value="1" onclick="pma_pw.value = ''; pma_pw2.value = ''; this.checked = true" />
-            <?php echo $GLOBALS['strNoPassword'] . "\n"; ?>
+            <?php echo  $GLOBALS['strNoPassword'] . "\n" ?>
         </td>
     </tr>
     <tr>
         <td>
             <input type="radio" name="nopass" value="0" checked="checked " />
-            <?php echo $GLOBALS['strPassword']; ?>&nbsp;:&nbsp;
+            <?php echo  $GLOBALS['strPassword'] ?>&nbsp;:&nbsp;
         </td>
         <td>
-            <input type="password" name="pma_pw" size="10" class="textfield" <?php echo $chg_evt_handler; ?>="nopass[1].checked = true" />
+            <input type="password" name="pma_pw" size="10" class="textfield" <?php echo  $chg_evt_handler ?>="nopass[1].checked = true" />
             &nbsp;&nbsp;
-            <?php echo $GLOBALS['strReType']; ?>&nbsp;:&nbsp;
-            <input type="password" name="pma_pw2" size="10" class="textfield" <?php echo $chg_evt_handler; ?>="nopass[1].checked = true" />
+            <?php echo  $GLOBALS['strReType'] ?>&nbsp;:&nbsp;
+            <input type="password" name="pma_pw2" size="10" class="textfield" <?php echo  $chg_evt_handler ?>="nopass[1].checked = true" />
         </td>
     </tr>
     <tr>