--- PageEdit.php (original)
+++ PageEdit.php (modified)
@@ -3,8 +3,8 @@
 // 홈페이지구축후 관리자 로그인후 페이지의 바로 html 수정을 할수 있게끔 하기위해서 
 ?>
 
-<?php if($HTTP_COOKIE_VARS[id_login_ok] && $_COOKIE[id_login_ok]){
-if($HTTP_COOKIE_VARS[id_login_ok]){$WebtingMemberLogin_id="$HTTP_COOKIE_VARS[id_login_ok]";}else if($_COOKIE[id_login_ok]){$WebtingMemberLogin_id="$_COOKIE[id_login_ok]";	}else{
+<?php if($HTTP_COOKIE_VARS['id_login_ok'] && $_COOKIE['id_login_ok']){
+if($HTTP_COOKIE_VARS['id_login_ok']){$WebtingMemberLogin_id="$HTTP_COOKIE_VARS['id_login_ok']";}else if($_COOKIE['id_login_ok']){$WebtingMemberLogin_id="$_COOKIE['id_login_ok']";	}else{
 echo ("<script language=javascript>
 window.alert('정상적으로 로그인 처리되어 있지 않은 상황입니다.\\n\\n문제1) 사용자의 인터넷 쿠키설정을 변경해주십시요.\\n\\n문제2) 웹팅의 DataBase에러일수 있습니다.\\n\\n문제1은 사용자가 해결해주셔야 하며 문제2는 윈도우를 재접속하시면 해결 처리 됩니다.');
 history.go(-1);
@@ -17,10 +17,10 @@
 
 /////////////////////////// 관리자 모드 호출 START //////////////////
 include "../db.php";
-$AdminChickTYyj= mysql_query("select * from member where no='1'",$db);
-$row_AdminChickTYyj= mysql_fetch_array($AdminChickTYyj);
-$BBSAdminloginKK="$row_AdminChickTYyj[id]";
-if($HTTP_COOKIE_VARS[id_login_ok]=="$BBSAdminloginKK"){}else{
+$AdminChickTYyj= mysqli_query("select * from member where no='1'",$db);
+$row_AdminChickTYyj= mysqli_fetch_array($AdminChickTYyj);
+$BBSAdminloginKK="$row_AdminChickTYyj['id']";
+if(isset($_COOKIE['id_login_ok']) && $_COOKIE['id_login_ok'] == $BBSAdminloginKK){}else{
   echo ("<script language=javascript>
                           window.alert('현페이지는 관리자만 진입할수 있습니다.');
                           window.self.close();
@@ -99,7 +99,7 @@
 
 <p align=center>
 <?php
-$CONTENT = eregi_replace("\\\\", "", $CONTENT);
+$CONTENT = preg_replace("\\\\", "", $CONTENT);
 ?>
 <textarea name='connent9' rows='42' cols='135'><?php echo $CONTENT?></textarea>
 </p>
@@ -120,7 +120,7 @@
 
 if($mode=="formok"){
 
-$CONTENTOK = eregi_replace("\\\\", "", $connent9);
+$CONTENTOK = preg_replace("\\\\", "", $connent9);
 
   if(!$connent9){
   echo ("<script language=javascript>