--- db_datadict.php (original)
+++ db_datadict.php (modified)
@@ -41,9 +41,9 @@
  */
 PMA_mysql_select_db($db);
 $sql    = 'SHOW TABLES FROM ' . PMA_backquote($db);
-$rowset = mysql_query($sql);
+$rowset = mysqli_query($sql);
 $count  = 0;
-while ($row = mysql_fetch_array($rowset)) {
+while ($row = mysqli_fetch_array($rowset)) {
     if (PMA_MYSQL_INT_VERSION >= 32303) {
         $myfieldname = 'Tables_in_' . htmlspecialchars($db);
     }
@@ -79,7 +79,7 @@
          $show_comment = '';
     } // end display comments
     if ($result) {
-         mysql_free_result($result);
+         mysqli_free_result($result);
     }
 
 
@@ -122,7 +122,7 @@
 
     } // end while
     if ($result) {
-        mysql_free_result($result);
+        mysqli_free_result($result);
     }
 
 
@@ -131,7 +131,7 @@
      */
     $local_query = 'SHOW FIELDS FROM ' . PMA_backquote($table);
     $result      = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, '', $err_url);
-    $fields_cnt  = mysql_num_rows($result);
+    $fields_cnt  = mysqli_num_rows($result);
 
     // Check if we can use Relations (Mike Beck)
     if (!empty($cfgRelation['relation'])) {
@@ -165,12 +165,12 @@
 <!-- TABLE INFORMATIONS -->
 <table width="100%" bordercolorlight="black" border="border" style="border-collapse: collapse;background-color: white">
 <tr>
-    <th width="50"><?php echo $strField; ?></th>
-    <th width="50"><?php echo $strType; ?></th>
-    <!--<th width="50"><?php echo $strAttr; ?></th>-->
-    <th width="50"><?php echo $strNull; ?></th>
-    <th width="50"><?php echo $strDefault; ?></th>
-    <!--<th width="50"><?php echo $strExtra; ?></th>-->
+    <th width="50"><?php echo  $strField ?></th>
+    <th width="50"><?php echo  $strType ?></th>
+    <!--<th width="50"><?php echo  $strAttr ?></th>-->
+    <th width="50"><?php echo  $strNull ?></th>
+    <th width="50"><?php echo  $strDefault ?></th>
+    <!--<th width="50"><?php echo  $strExtra ?></th>-->
     <?php
     echo "\n";
     if ($have_rel) {
@@ -191,23 +191,23 @@
         $type             = $row['Type'];
         // reformat mysql query output - staybyte - 9. June 2001
         // loic1: set or enum types: slashes single quotes inside options
-        if (eregi('^(set|enum)\((.+)\)$', $type, $tmp)) {
-            $tmp[2]       = substr(ereg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
+        if (preg_match('^(set|enum)\((.+)\)$', $type, $tmp)) {
+            $tmp[2]       = substr(preg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
             $type         = $tmp[1] . '(' . str_replace(',', ', ', $tmp[2]) . ')';
             $type_nowrap  = '';
         } else {
             $type_nowrap  = ' nowrap="nowrap"';
         }
-        $type             = eregi_replace('BINARY', '', $type);
-        $type             = eregi_replace('ZEROFILL', '', $type);
-        $type             = eregi_replace('UNSIGNED', '', $type);
+        $type             = preg_replace('BINARY', '', $type);
+        $type             = preg_replace('ZEROFILL', '', $type);
+        $type             = preg_replace('UNSIGNED', '', $type);
         if (empty($type)) {
             $type         = '&nbsp;';
         }
 
-        $binary           = eregi('BINARY', $row['Type'], $test);
-        $unsigned         = eregi('UNSIGNED', $row['Type'], $test);
-        $zerofill         = eregi('ZEROFILL', $row['Type'], $test);
+        $binary           = preg_match('BINARY', $row['Type'], $test);
+        $unsigned         = preg_match('UNSIGNED', $row['Type'], $test);
+        $zerofill         = preg_match('ZEROFILL', $row['Type'], $test);
         $strAttribute     = '&nbsp;';
         if ($binary) {
             $strAttribute = 'BINARY';
@@ -239,11 +239,11 @@
         }
         ?>
     </td>
-    <td width="50" class="print"<?php echo $type_nowrap; ?>><?php echo $type; ?><bdo dir="ltr"></bdo></td>
-    <!--<td width="50" bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $strAttribute; ?></td>-->
-    <td width="50" class="print"><?php echo (($row['Null'] == '') ? $strNo : $strYes); ?>&nbsp;</td>
+    <td width="50" class="print"<?php echo  $type_nowrap ?>><?php echo  $type ?><bdo dir="ltr"></bdo></td>
+    <!--<td width="50" bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $strAttribute ?></td>-->
+    <td width="50" class="print"><?php echo  (($row['Null'] == '') ? $strNo : $strYes) ?>&nbsp;</td>
     <td width="50" class="print" nowrap="nowrap"><?php if (isset($row['Default'])) echo $row['Default']; ?>&nbsp;</td>
-    <!--<td width="50" bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $row['Extra']; ?>&nbsp;</td>-->
+    <!--<td width="50" bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $row['Extra'] ?>&nbsp;</td>-->
         <?php
         echo "\n";
         if ($have_rel) {
@@ -264,7 +264,7 @@
 </tr>
         <?php
     } // end while
-    mysql_free_result($result);
+    mysqli_free_result($result);
 
     echo "\n";
     ?>