--- db_details_common.php (original)
+++ db_details_common.php (modified)
@@ -54,7 +54,7 @@
         ?>
 <script type="text/javascript" language="javascript1.2">
 <!--
-window.parent.frames['nav'].location.replace('./left.php?lang=<?php echo $lang; ?> &convcharset=<?php echo $convcharset; ?>&server=<?php echo $server; ?>&db=<?php echo urlencode($db); ?>');
+window.parent.frames['nav'].location.replace('./left.php?lang=<?php echo  $lang ?> &convcharset=<?php echo  $convcharset ?>&server=<?php echo  $server ?>&db=<?php echo  urlencode($db) ?>');
 //-->
 </script>
         <?php