--- db_stats.php (original)
+++ db_stats.php (modified)
@@ -83,7 +83,7 @@
 if ($server > 0) {
     // Get the valid databases list
     $num_dbs = count($dblist);
-    $dbs     = @mysql_list_dbs() or PMA_mysqlDie('', 'mysql_list_dbs()', '', 'main.php?lang' . $lang . '&amp;server=' . $server);
+    $dbs     = @mysqli_list_dbs() or PMA_mysqlDie('', 'mysqli_list_dbs()', '', 'main.php?lang' . $lang . '&amp;server=' . $server);
     if ($dbs) {
         while ($a_db = PMA_mysql_fetch_object($dbs)) {
             if (!$num_dbs) {
@@ -92,7 +92,7 @@
                 $true_dblist[$a_db->Database] = '';
             }
         } // end while
-        mysql_free_result($dbs);
+        mysqli_free_result($dbs);
     } // end if
     if ($num_dbs && empty($true_dblist)) {
         $dblist = array();
@@ -123,16 +123,16 @@
  */
 ?>
 <h1 align="center">
-    <?php echo $strDatabasesStats; ?>
+    <?php echo  $strDatabasesStats ?>
 </h1>
-<table align="center" border="<?php echo $cfg['Border']; ?>" cellpadding="5">
+<table align="center" border="<?php echo  $cfg['Border'] ?>" cellpadding="5">
 <tr>
-    <th align="<?php echo $cell_align_left; ?>"><big><?php echo $strHost . ' :'; ?></big></th>
-    <th align="<?php echo $cell_align_left; ?>"><big><?php echo $cfg['Server']['host']; ?></big></th>
+    <th align="<?php echo  $cell_align_left ?>"><big><?php echo  $strHost . ' :' ?></big></th>
+    <th align="<?php echo  $cell_align_left ?>"><big><?php echo  $cfg['Server']['host'] ?></big></th>
 </tr>
 <tr>
-    <th align="<?php echo $cell_align_left; ?>"><big><?php echo $strGenTime . ' :'; ?></big></th>
-    <th align="<?php echo $cell_align_left; ?>"><big><?php echo PMA_localisedDate(); ?></big></th>
+    <th align="<?php echo  $cell_align_left ?>"><big><?php echo  $strGenTime . ' :' ?></big></th>
+    <th align="<?php echo  $cell_align_left ?>"><big><?php echo  PMA_localisedDate() ?></big></th>
 </tr>
 </table>
 <br /><br />
@@ -185,37 +185,37 @@
     }
     ?>
 <form action="db_stats.php" name="dbStatsForm">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-
-    <table align="center" border="<?php echo $cfg['Border']; ?>">
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+
+    <table align="center" border="<?php echo  $cfg['Border'] ?>">
     <tr>
         <th>&nbsp;</th>
         <th>
             &nbsp;
-            <a href="<?php echo $common_url . '&amp;sort_by=db_name&amp;sort_order=' . $url_sort[0]['order']; ?>">
-                <?php echo $strDatabase . $url_sort[0]['img_tag']; ?></a>&nbsp;
-        </th>
-        <th>
-            &nbsp;
-            <a href="<?php echo $common_url . '&amp;sort_by=tbl_cnt&amp;sort_order=' . $url_sort[1]['order']; ?>">
-                <?php echo $strNumTables . $url_sort[1]['img_tag']; ?></a>&nbsp;
-        </th>
-        <th>
-            &nbsp;
-            <a href="<?php echo $common_url . '&amp;sort_by=data_sz&amp;sort_order=' . $url_sort[2]['order']; ?>">
-                <?php echo $strData . $url_sort[2]['img_tag']; ?></a>&nbsp;
-        </th>
-        <th>
-            &nbsp;
-            <a href="<?php echo $common_url . '&amp;sort_by=idx_sz&amp;sort_order=' . $url_sort[3]['order']; ?>">
-                <?php echo $strIndexes . $url_sort[3]['img_tag']; ?></a>&nbsp;
-        </th>
-        <th>
-            &nbsp;
-            <a href="<?php echo $common_url . '&amp;sort_by=tot_sz&amp;sort_order=' . $url_sort[4]['order']; ?>">
-                <?php echo $strTotalUC . $url_sort[4]['img_tag']; ?></a>&nbsp;
+            <a href="<?php echo  $common_url . '&amp;sort_by=db_name&amp;sort_order=' . $url_sort[0]['order'] ?>">
+                <?php echo  $strDatabase . $url_sort[0]['img_tag'] ?></a>&nbsp;
+        </th>
+        <th>
+            &nbsp;
+            <a href="<?php echo  $common_url . '&amp;sort_by=tbl_cnt&amp;sort_order=' . $url_sort[1]['order'] ?>">
+                <?php echo  $strNumTables . $url_sort[1]['img_tag'] ?></a>&nbsp;
+        </th>
+        <th>
+            &nbsp;
+            <a href="<?php echo  $common_url . '&amp;sort_by=data_sz&amp;sort_order=' . $url_sort[2]['order'] ?>">
+                <?php echo  $strData . $url_sort[2]['img_tag'] ?></a>&nbsp;
+        </th>
+        <th>
+            &nbsp;
+            <a href="<?php echo  $common_url . '&amp;sort_by=idx_sz&amp;sort_order=' . $url_sort[3]['order'] ?>">
+                <?php echo  $strIndexes . $url_sort[3]['img_tag'] ?></a>&nbsp;
+        </th>
+        <th>
+            &nbsp;
+            <a href="<?php echo  $common_url . '&amp;sort_by=tot_sz&amp;sort_order=' . $url_sort[4]['order'] ?>">
+                <?php echo  $strTotalUC . $url_sort[4]['img_tag'] ?></a>&nbsp;
         </th>
     </tr>
     <?php
@@ -234,8 +234,8 @@
 
         // Number of tables
         if ($tables) {
-            $dbs_array[$db][0] = mysql_numrows($tables);
-            mysql_free_result($tables);
+            $dbs_array[$db][0] = mysqli_numrows($tables);
+            mysqli_free_result($tables);
         } else {
             $dbs_array[$db][0] = 0;
         }
@@ -250,7 +250,7 @@
             $local_query = 'SHOW TABLE STATUS FROM ' . PMA_backquote($db);
             $result      = @PMA_mysql_query($local_query);
             // needs the "@" below otherwise, warnings in case of special DB names
-            if ($result && @mysql_num_rows($result)) {
+            if ($result && @mysqli_num_rows($result)) {
                 while ($row = PMA_mysql_fetch_array($result)) {
                     $dbs_array[$db][1] += $row['Data_length'];
                     $dbs_array[$db][2] += $row['Index_length'];
@@ -259,12 +259,12 @@
                 $total_array[1]        += $dbs_array[$db][1];
                 $total_array[2]        += $dbs_array[$db][2];
                 $total_array[3]        += $dbs_array[$db][3];
-                mysql_free_result($result);
+                mysqli_free_result($result);
             } // end if
         } // end if MySQL 3.23.03+
 
     } // end for
-    mysql_close();
+    mysqli_close();
 
     // Sorts the dbs arrays
     uksort($dbs_array, 'PMA_dbCmp');
@@ -345,7 +345,7 @@
  */
 else {
     ?>
-<p align="center"><big>&nbsp;&nbsp;<?php echo $strNoDatabases; ?></big></p>
+<p align="center"><big>&nbsp;&nbsp;<?php echo  $strNoDatabases ?></big></p>
     <?php
 } // end if ($num_dbs == 0)
 echo "\n";