--- footer.inc.php (original)
+++ footer.inc.php (modified)
@@ -9,10 +9,10 @@
  * Close MySql non-persistent connections
  */
 if (isset($GLOBALS['dbh']) && $GLOBALS['dbh']) {
-    @mysql_close($GLOBALS['dbh']);
+    @mysqli_close($GLOBALS['dbh']);
 }
 if (isset($GLOBALS['userlink']) && $GLOBALS['userlink']) {
-    @mysql_close($GLOBALS['userlink']);
+    @mysqli_close($GLOBALS['userlink']);
 }
 ?>
 