--- left.php (original)
+++ left.php (modified)
@@ -59,11 +59,11 @@
 ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $available_languages[$lang][2]; ?>" lang="<?php echo $available_languages[$lang][2]; ?>" dir="<?php echo $text_dir; ?>">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo  $available_languages[$lang][2] ?>" lang="<?php echo  $available_languages[$lang][2] ?>" dir="<?php echo  $text_dir ?>">
 
 <head>
     <title>phpMyAdmin</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=<?php echo $charset; ?>" />
+    <meta http-equiv="Content-Type" content="text/html; charset=<?php echo  $charset ?>" />
     <base<?php if (!empty($cfg['PmaAbsoluteUri'])) echo ' href="' . $cfg['PmaAbsoluteUri'] . '"'; ?> target="phpmain" />
 <?php
 // Expandable/collapsible databases list is only used if there is more than one
@@ -96,22 +96,22 @@
             }
         } // end if... else if...
     } // end if
-    var fontFamily  = '<?php echo $left_font_family; ?>';
-    var fontSize    = '<?php echo $font_size; ?>';
-    var fontBig     = '<?php echo $font_bigger; ?>';
-    var fontSmall   = '<?php echo $font_smaller; ?>';
-    var isServer    = <?php echo ($server > 0) ? 'true' : 'false'; ?>;
+    var fontFamily  = '<?php echo  $left_font_family ?>';
+    var fontSize    = '<?php echo  $font_size ?>';
+    var fontBig     = '<?php echo  $font_bigger ?>';
+    var fontSmall   = '<?php echo  $font_smaller ?>';
+    var isServer    = <?php echo  ($server > 0) ? 'true' : 'false' ?>;
     //-->
     </script>
     <script src="libraries/left.js" type="text/javascript" language="javascript1.2"></script>
     <noscript>
         <style type="text/css">
         <!--
-        div {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-        .heada {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-        .headaCnt {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_smaller; ?>; color: #000000}
-        .parent {font-family: <?php echo $left_font_family; ?>; color: #000000; text-decoration: none}
-        .child {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_smaller; ?>; color: #333399; text-decoration: none}
+        div {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+        .heada {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+        .headaCnt {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_smaller ?>; color: #000000}
+        .parent {font-family: <?php echo  $left_font_family ?>; color: #000000; text-decoration: none}
+        .child {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_smaller ?>; color: #333399; text-decoration: none}
         .item, .item:active, .item:hover, .tblItem, .tblItem:active {color: #333399; text-decoration: none}
         .tblItem:hover {color: #FF0000; text-decoration: underline}
         //-->
@@ -120,7 +120,7 @@
 
     <style type="text/css">
     <!--
-    body {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>}
+    body {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>}
     //-->
     </style>
     <?php
@@ -131,13 +131,13 @@
     ?>
     <style type="text/css">
     <!--
-    body {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>}
-    div {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-    .heada {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-    .headaCnt {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_smaller; ?>; color: #000000}
-    .parent {font-family: <?php echo $left_font_family; ?>; color: #000000; text-decoration: none}
-    .child {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_smaller; ?>; color: #333399; text-decoration: none}
-    .item, .item:active, .item:hover, .tblItem, .tblItem:active {font-size: <?php echo $font_smaller; ?>; color: #333399; text-decoration: none}
+    body {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>}
+    div {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+    .heada {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+    .headaCnt {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_smaller ?>; color: #000000}
+    .parent {font-family: <?php echo  $left_font_family ?>; color: #000000; text-decoration: none}
+    .child {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_smaller ?>; color: #333399; text-decoration: none}
+    .item, .item:active, .item:hover, .tblItem, .tblItem:active {font-size: <?php echo  $font_smaller ?>; color: #333399; text-decoration: none}
     .tblItem:hover {color: #FF0000; text-decoration: underline}
     //-->
     </style>
@@ -149,13 +149,13 @@
     ?>
     <style type="text/css">
     <!--
-    body    {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>}
-    div     {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-    input   {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>}
-    select  {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; background-color: #ffffff; color: #000000}
-    .heada  {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-    .parent {font-family: <?php echo $left_font_family; ?>; color: #000000; text-decoration: none}
-    .item, .item:active, .item:hover, .tblItem, .tblItem:active {font-family: <?php echo $left_font_family; ?>; font-size: <?php echo $font_smaller; ?>; color: #333399; text-decoration: none}
+    body    {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>}
+    div     {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+    input   {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>}
+    select  {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; background-color: #ffffff; color: #000000}
+    .heada  {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+    .parent {font-family: <?php echo  $left_font_family ?>; color: #000000; text-decoration: none}
+    .item, .item:active, .item:hover, .tblItem, .tblItem:active {font-family: <?php echo  $left_font_family ?>; font-size: <?php echo  $font_smaller ?>; color: #333399; text-decoration: none}
     .tblItem:hover {color: #FF0000; text-decoration: underline}
     //-->
     </style>
@@ -166,7 +166,7 @@
 ?>
 </head>
 
-<body bgcolor="<?php echo $cfg['LeftBgColor']; ?>">
+<body bgcolor="<?php echo  $cfg['LeftBgColor'] ?>">
 
 <?php
 if ($cfg['LeftDisplayLogo']) {
@@ -179,7 +179,7 @@
 ?>
     <!-- Link to the welcome page -->
     <div id="el1Parent" class="parent" style="margin-bottom: 5px">
-        <nobr><a class="item" href="main.php?lang=<?php echo $lang; ?>&amp;convcharset=<?php echo $convcharset; ?>&amp;server=<?php echo $server; ?>"><span class="heada"><b><?php echo $strHome; ?></b></span></a></nobr>
+        <nobr><a class="item" href="main.php?lang=<?php echo  $lang ?>&amp;convcharset=<?php echo  $convcharset ?>&amp;server=<?php echo  $server ?>"><span class="heada"><b><?php echo  $strHome ?></b></span></a></nobr>
     </div>
 
 
@@ -218,7 +218,7 @@
             $selected_db = $j;
         }
         $tables              = @PMA_mysql_list_tables($db);
-        $num_tables          = ($tables) ? @mysql_numrows($tables) : 0;
+        $num_tables          = ($tables) ? @mysqli_numrows($tables) : 0;
         $common_url_query    = 'lang=' . $lang
                              . '&amp;convcharset=' . $convcharset
                              . '&amp;server=' . $server
@@ -253,8 +253,8 @@
             if (!empty($num_tables)) {
                 echo "\n";
                 ?>
-        <nobr><a class="item" href="<?php echo $cfg['DefaultTabDatabase']; ?>?<?php echo $common_url_query; ?>" onclick="if (capable) {expandBase('el<?php echo $j; ?>', true); return false} else {return true}">
-            <img name="imEx" id="el<?php echo $j; ?>Img" src="images/plus.gif" border="0" width="9" height="9" alt="+" /></a>
+        <nobr><a class="item" href="<?php echo  $cfg['DefaultTabDatabase'] ?>?<?php echo  $common_url_query ?>" onclick="if (capable) {expandBase('el<?php echo  $j ?>', true); return false} else {return true}">
+            <img name="imEx" id="el<?php echo  $j ?>Img" src="images/plus.gif" border="0" width="9" height="9" alt="+" /></a>
                 <?php
             } else {
                 echo "\n";
@@ -264,11 +264,11 @@
             }
             echo "\n";
             ?>
-        <a class="item" href="<?php echo $cfg['DefaultTabDatabase']; ?>?<?php echo $common_url_query; ?>" onclick="if (capable) {expandBase('el<?php echo $j; ?>', false)}">
-            <span class="heada"><?php echo htmlspecialchars($db); ?><bdo dir="<?php echo($text_dir); ?>">&nbsp;&nbsp;</bdo></span><span class="headaCnt">(<?php echo $num_tables_disp; ?>)</span></a></nobr>
+        <a class="item" href="<?php echo  $cfg['DefaultTabDatabase'] ?>?<?php echo  $common_url_query ?>" onclick="if (capable) {expandBase('el<?php echo  $j ?>', false)}">
+            <span class="heada"><?php echo  htmlspecialchars($db) ?><bdo dir="<?php echo($text_dir); ?>">&nbsp;&nbsp;</bdo></span><span class="headaCnt">(<?php echo  $num_tables_disp ?>)</span></a></nobr>
     </div>
 
-    <div id="el<?php echo $j;?>Child" class="child" style="margin-bottom: 5px"<?php echo $on_mouse; ?>>
+    <div id="el<?php echo  $j ?>Child" class="child" style="margin-bottom: 5px"<?php echo  $on_mouse ?>>
 
             <?php
             // Displays the list of tables from the current database
@@ -280,10 +280,10 @@
                 echo "\n";
                 ?>
         <nobr><img src="images/spacer.gif" border="0" width="9" height="9" alt="" />
-        <a target="phpmain" href="sql.php?<?php echo $common_url_query; ?>&amp;table=<?php echo urlencode($table); ?>&amp;sql_query=<?php echo urlencode('SELECT * FROM ' . PMA_backquote($table)); ?>&amp;pos=0&amp;goto=<?php echo $cfg['DefaultTabTable']; ?>" title="<?php echo $strBrowse . ': ' . htmlspecialchars($table); ?>">
-            <img src="images/browse.gif" width="8" height="8" border="0" alt="<?php echo $strBrowse . ': ' . htmlspecialchars($table); ?>" /></a><bdo dir="<?php echo $text_dir; ?>">&nbsp;</bdo>
-        <a class="tblItem" id="tbl_<?php echo md5($table); ?>" title="<?php echo $url_title; ?>" target="phpmain" href="<?php echo $cfg['DefaultTabTable']; ?>?<?php echo $common_url_query; ?>&amp;table=<?php echo urlencode($table); ?>">
-            <?php echo htmlspecialchars($table); ?></a></nobr><br />
+        <a target="phpmain" href="sql.php?<?php echo  $common_url_query ?>&amp;table=<?php echo  urlencode($table) ?>&amp;sql_query=<?php echo  urlencode('SELECT * FROM ' . PMA_backquote($table)) ?>&amp;pos=0&amp;goto=<?php echo  $cfg['DefaultTabTable'] ?>" title="<?php echo  $strBrowse . ': ' . htmlspecialchars($table) ?>">
+            <img src="images/browse.gif" width="8" height="8" border="0" alt="<?php echo  $strBrowse . ': ' . htmlspecialchars($table) ?>" /></a><bdo dir="<?php echo  $text_dir ?>">&nbsp;</bdo>
+        <a class="tblItem" id="tbl_<?php echo  md5($table) ?>" title="<?php echo  $url_title ?>" target="phpmain" href="<?php echo  $cfg['DefaultTabTable'] ?>?<?php echo  $common_url_query ?>&amp;table=<?php echo  urlencode($table) ?>">
+            <?php echo  htmlspecialchars($table) ?></a></nobr><br />
                 <?php
             } // end for $t (tables list)
             echo "\n";
@@ -371,7 +371,7 @@
       nsShowAll();
       nsArrangeList();
     }
-    var expandedDb = '<?php echo (empty($selected_db)) ? '' : 'el' . $selected_db . 'Child'; ?>';
+    var expandedDb = '<?php echo  (empty($selected_db)) ? '' : 'el' . $selected_db . 'Child' ?>';
     //-->
     </script>
         <?php
@@ -385,7 +385,7 @@
 else if ($num_dbs == 1) {
     $db                  = $dblist[0];
     $tables              = @PMA_mysql_list_tables($db);
-    $num_tables          = ($tables) ? @mysql_numrows($tables) : 0;
+    $num_tables          = ($tables) ? @mysqli_numrows($tables) : 0;
     $common_url_query    = 'lang=' . $lang
                          . '&amp;server=' . $server
                          . '&amp;db=' . urlencode($db);
@@ -410,8 +410,8 @@
     echo "\n";
     ?>
     <div id="el2Parent" class="parent">
-        <nobr><a class="item" href="<?php echo $cfg['DefaultTabDatabase']; ?>?<?php echo $common_url_query; ?>">
-            <span class="heada"><?php echo htmlspecialchars($db); ?><bdo dir="<?php echo($text_dir); ?>">&nbsp;&nbsp;</bdo></span><span class="headaCnt">(<?php echo $num_tables_disp; ?>)</span></a></nobr>
+        <nobr><a class="item" href="<?php echo  $cfg['DefaultTabDatabase'] ?>?<?php echo  $common_url_query ?>">
+            <span class="heada"><?php echo  htmlspecialchars($db) ?><bdo dir="<?php echo($text_dir); ?>">&nbsp;&nbsp;</bdo></span><span class="headaCnt">(<?php echo  $num_tables_disp ?>)</span></a></nobr>
     </div>
     <div id="el2Child" class="child" style="margin-bottom: 5px">
     <?php
@@ -423,10 +423,10 @@
                    : '';
         echo "\n";
         ?>
-        <nobr><a target="phpmain" href="sql.php?<?php echo $common_url_query; ?>&amp;table=<?php echo urlencode($table); ?>&amp;sql_query=<?php echo urlencode('SELECT * FROM ' . PMA_backquote($table)); ?>&amp;pos=0&amp;goto=<?php echo $cfg['DefaultTabTable']; ?>" title="<?php echo $strBrowse . ': ' . htmlspecialchars($table); ?>">
-                  <img src="images/browse.gif" width="8" height="8" border="0" alt="<?php echo $strBrowse . ': ' . htmlspecialchars($table); ?>" /></a><bdo dir="<?php echo $text_dir; ?>">&nbsp;</bdo>
-              <a class="tblItem" id="tbl_<?php echo md5($table); ?>" title="<?php echo $url_title; ?>" target="phpmain" href="<?php echo $cfg['DefaultTabTable']; ?>?<?php echo $common_url_query; ?>&amp;table=<?php echo urlencode($table); ?>">
-                  <?php echo htmlspecialchars($table); ?></a></nobr><br />
+        <nobr><a target="phpmain" href="sql.php?<?php echo  $common_url_query ?>&amp;table=<?php echo  urlencode($table) ?>&amp;sql_query=<?php echo  urlencode('SELECT * FROM ' . PMA_backquote($table)) ?>&amp;pos=0&amp;goto=<?php echo  $cfg['DefaultTabTable'] ?>" title="<?php echo  $strBrowse . ': ' . htmlspecialchars($table) ?>">
+                  <img src="images/browse.gif" width="8" height="8" border="0" alt="<?php echo  $strBrowse . ': ' . htmlspecialchars($table) ?>" /></a><bdo dir="<?php echo  $text_dir ?>">&nbsp;</bdo>
+              <a class="tblItem" id="tbl_<?php echo  md5($table) ?>" title="<?php echo  $url_title ?>" target="phpmain" href="<?php echo  $cfg['DefaultTabTable'] ?>?<?php echo  $common_url_query ?>&amp;table=<?php echo  urlencode($table) ?>">
+                  <?php echo  htmlspecialchars($table) ?></a></nobr><br />
         <?php
     } // end for $j (tables list)
     echo "\n";
@@ -452,10 +452,10 @@
  * Close MySql connections
  */
 if (isset($dbh) && $dbh) {
-    @mysql_close($dbh);
+    @mysqli_close($dbh);
 }
 if (isset($userlink) && $userlink) {
-    @mysql_close($userlink);
+    @mysqli_close($userlink);
 }
 
 