--- main.php (original)
+++ main.php (modified)
@@ -40,7 +40,7 @@
     ?>
 <script type="text/javascript" language="javascript1.2">
 <!--
-window.parent.frames['nav'].location.replace('./left.php?lang=<?php echo $lang; ?>&convcharset=<?php echo $convcharset; ?>&server=<?php echo $server; ?>');
+window.parent.frames['nav'].location.replace('./left.php?lang=<?php echo  $lang ?>&convcharset=<?php echo  $convcharset ?>&server=<?php echo  $server ?>');
 //-->
 </script>
     <?php
@@ -52,7 +52,7 @@
  * Displays the welcome message and the server informations
  */
 ?>
-<h1><?php echo sprintf($strWelcome, ' phpMyAdmin ' . PMA_VERSION); ?></h1>
+<h1><?php echo  sprintf($strWelcome, ' phpMyAdmin ' . PMA_VERSION) ?></h1>
 
 <?php
 // Don't display server info if $server == 0 (no server selected)
@@ -74,12 +74,12 @@
     // }
     $local_query             = 'SELECT VERSION() as version, USER() as user';
     $res                     = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, FALSE, '');
-    $mysql_cur_user_and_host = PMA_mysql_result($res, 0, 'user');
-    $mysql_cur_user          = substr($mysql_cur_user_and_host, 0, strpos($mysql_cur_user_and_host, '@'));
+    $mysqli_cur_user_and_host = PMA_mysql_result($res, 0, 'user');
+    $mysqli_cur_user          = substr($mysqli_cur_user_and_host, 0, strpos($mysqli_cur_user_and_host, '@'));
 
     $full_string     = str_replace('%pma_s1%', PMA_mysql_result($res, 0, 'version'), $strMySQLServerProcess);
     $full_string     = str_replace('%pma_s2%', $server_info, $full_string);
-    $full_string     = str_replace('%pma_s3%', $mysql_cur_user_and_host, $full_string);
+    $full_string     = str_replace('%pma_s3%', $mysqli_cur_user_and_host, $full_string);
 
     echo '<p><b>' . $full_string . '</b></p><br />' . "\n";
 } // end if
@@ -108,7 +108,7 @@
 <!-- MySQL servers choice form -->
 <table>
 <tr>
-    <th><?php echo $strServerChoice; ?></th>
+    <th><?php echo  $strServerChoice ?></th>
 </tr>
 <tr>
     <td>
@@ -151,9 +151,9 @@
     } // end while
     ?>
             </select>
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="submit" value="<?php echo $strGo; ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="submit" value="<?php echo  $strGo ?>" />
         </form>
     </td>
 </tr>
@@ -180,7 +180,7 @@
     $is_reload_priv  = FALSE;
     $is_superuser    = @PMA_mysql_query('USE mysql', $userlink);
     if ($dbh) {
-        $local_query = 'SELECT Create_priv, Process_priv, Reload_priv FROM mysql.user WHERE User = \'' . PMA_sqlAddslashes($mysql_cur_user) . '\'';
+        $local_query = 'SELECT Create_priv, Process_priv, Reload_priv FROM mysql.user WHERE User = \'' . PMA_sqlAddslashes($mysqli_cur_user) . '\'';
         $rs_usr      = PMA_mysql_query($local_query, $dbh); // Debug: or PMA_mysqlDie('', $local_query, FALSE);
         if ($rs_usr) {
             while ($result_usr = PMA_mysql_fetch_array($rs_usr)) {
@@ -194,7 +194,7 @@
                     $is_reload_priv  = ($result_usr['Reload_priv'] == 'Y');
                 }
             } // end while
-            mysql_free_result($rs_usr);
+            mysqli_free_result($rs_usr);
         } // end if
     } // end if
 
@@ -202,32 +202,32 @@
     // the first inexistant db name that we find, in most cases it's probably
     // the one he just dropped :)
     if (!$is_create_priv) {
-        $local_query = 'SELECT DISTINCT Db FROM mysql.db WHERE Create_priv = \'Y\' AND User = \'' . PMA_sqlAddslashes($mysql_cur_user) . '\'';
+        $local_query = 'SELECT DISTINCT Db FROM mysql.db WHERE Create_priv = \'Y\' AND User = \'' . PMA_sqlAddslashes($mysqli_cur_user) . '\'';
         $rs_usr      = PMA_mysql_query($local_query, $dbh); // Debug: or PMA_mysqlDie('', $local_query, FALSE);
         if ($rs_usr) {
             $re0     = '(^|(\\\\\\\\)+|[^\])'; // non-escaped wildcards
             $re1     = '(^|[^\])(\\\)+';       // escaped wildcards
             while ($row = PMA_mysql_fetch_array($rs_usr)) {
-                if (ereg($re0 . '(%|_)', $row['Db'])
-                    || (!PMA_mysql_select_db(ereg_replace($re1 . '(%|_)', '\\1\\3', $row['Db']), $userlink) && @mysql_errno() != 1044)) {
-                    $db_to_create   = ereg_replace($re0 . '%', '\\1...', ereg_replace($re0 . '_', '\\1?', $row['Db']));
-                    $db_to_create   = ereg_replace($re1 . '(%|_)', '\\1\\3', $db_to_create);
+                if (preg_match($re0 . '(%|_)', $row['Db'])
+                    || (!PMA_mysql_select_db(preg_replace($re1 . '(%|_)', '\\1\\3', $row['Db']), $userlink) && @mysqli_errno() != 1044)) {
+                    $db_to_create   = preg_replace($re0 . '%', '\\1...', preg_replace($re0 . '_', '\\1?', $row['Db']));
+                    $db_to_create   = preg_replace($re1 . '(%|_)', '\\1\\3', $db_to_create);
                     $is_create_priv = TRUE;
                     break;
                 } // end if
             } // end while
-            mysql_free_result($rs_usr);
+            mysqli_free_result($rs_usr);
         } // end if
         else if (PMA_MYSQL_INT_VERSION >= 32304) {
             // Finally, let's try to get the user's privileges by using SHOW
             // GRANTS...
             // Maybe we'll find a little CREATE priv there :)
-            $local_query = 'SHOW GRANTS FOR ' . $mysql_cur_user_and_host;
+            $local_query = 'SHOW GRANTS FOR ' . $mysqli_cur_user_and_host;
             $rs_usr      = PMA_mysql_query($local_query, $dbh);
             if (!$rs_usr) {
                 // OK, now we'd have to guess the user's hostname, but we
                 // only try out the 'username'@'%' case.
-                $local_query = 'SHOW GRANTS FOR ' . $mysql_cur_user;
+                $local_query = 'SHOW GRANTS FOR ' . $mysqli_cur_user;
                 $rs_usr      = PMA_mysql_query($local_query, $dbh);
             }
             if ($rs_usr) {
@@ -242,9 +242,9 @@
                             $db_to_create   = '';
                             break;
                         } // end if
-                        else if (ereg($re0 . '%|_', $show_grants_dbname) || !PMA_mysql_select_db($show_grants_dbname, $userlink) && @mysql_errno() != 1044) {
-                            $db_to_create = ereg_replace($re0 . '%', '\\1...', ereg_replace($re0 . '_', '\\1?', $show_grants_dbname));
-                            $db_to_create = ereg_replace($re1 . '(%|_)', '\\1\\3', $db_to_create);
+                        else if (preg_match($re0 . '%|_', $show_grants_dbname) || !PMA_mysql_select_db($show_grants_dbname, $userlink) && @mysqli_errno() != 1044) {
+                            $db_to_create = preg_replace($re0 . '%', '\\1...', preg_replace($re0 . '_', '\\1?', $show_grants_dbname));
+                            $db_to_create = preg_replace($re1 . '(%|_)', '\\1\\3', $db_to_create);
                             $is_create_priv     = TRUE;
                             break;
                         } // end elseif
@@ -252,7 +252,7 @@
                 } // end while
                 unset($show_grants_dbname);
                 unset($show_grants_str);
-                mysql_free_result($rs_usr);
+                mysqli_free_result($rs_usr);
             } // end if
         } // end elseif
     } // end if
@@ -282,7 +282,7 @@
         || $cfg['Server']['auth_type'] != 'config') {
         ?>
     <!-- MySQL server related links -->
-    <td valign="top" align="<?php echo $cell_align_left; ?>">
+    <td valign="top" align="<?php echo  $cell_align_left ?>">
         <table>
         <tr>
             <th colspan="2">&nbsp;&nbsp;MySQL</th>
@@ -294,16 +294,16 @@
             ?>
         <!-- db creation form -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
             <td>
             <form method="post" action="db_create.php">
-                <?php echo $strCreateNewDatabase . '&nbsp;' . PMA_showMySQLDocu('Reference', 'CREATE_DATABASE'); ?><br />
-                <input type="hidden" name="server" value="<?php echo $server; ?>" />
-                <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-                <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
+                <?php echo  $strCreateNewDatabase . '&nbsp;' . PMA_showMySQLDocu('Reference', 'CREATE_DATABASE') ?><br />
+                <input type="hidden" name="server" value="<?php echo  $server ?>" />
+                <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+                <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
                 <input type="hidden" name="reload" value="1" />
-                <input type="text" name="db" value="<?php echo $db_to_create; ?>" maxlength="64" class="textfield" />
-                <input type="submit" value="<?php echo $strCreate; ?>" />
+                <input type="text" name="db" value="<?php echo  $db_to_create ?>" maxlength="64" class="textfield" />
+                <input type="submit" value="<?php echo  $strCreate ?>" />
             </form>
             </td>
         </tr>
@@ -313,10 +313,10 @@
             ?>
         <!-- db creation no privileges message -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <?php echo $strCreateNewDatabase . ':&nbsp;' . PMA_showMySQLDocu('Reference', 'CREATE_DATABASE'); ?><br />
-                <?php echo '<i>' . $strNoPrivileges .'</i>'; ?><br />
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <?php echo  $strCreateNewDatabase . ':&nbsp;' . PMA_showMySQLDocu('Reference', 'CREATE_DATABASE') ?><br />
+                <?php echo  '<i>' . $strNoPrivileges .'</i>' ?><br />
             </td>
         </tr>
             <?php
@@ -331,11 +331,11 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="sql.php?<?php echo $common_url_query; ?>&amp;db=mysql&amp;sql_query=<?php echo urlencode('SHOW STATUS'); ?>&amp;goto=main.php">
-                    <?php echo $strMySQLShowStatus; ?></a>&nbsp;
-                <?php echo PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_STATUS') . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="sql.php?<?php echo  $common_url_query ?>&amp;db=mysql&amp;sql_query=<?php echo  urlencode('SHOW STATUS') ?>&amp;goto=main.php">
+                    <?php echo  $strMySQLShowStatus ?></a>&nbsp;
+                <?php echo  PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_STATUS') . "\n" ?>
             </td>
         </tr>
             <?php
@@ -344,11 +344,11 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="sql.php?<?php echo $common_url_query; ?>&amp;db=mysql&amp;sql_query=<?php echo urlencode('SHOW VARIABLES'); ?>&amp;goto=main.php">
-                <?php echo $strMySQLShowVars;?></a>&nbsp;
-                <?php echo PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_VARIABLES') . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="sql.php?<?php echo  $common_url_query ?>&amp;db=mysql&amp;sql_query=<?php echo  urlencode('SHOW VARIABLES') ?>&amp;goto=main.php">
+                <?php echo  $strMySQLShowVars ?></a>&nbsp;
+                <?php echo  PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_VARIABLES') . "\n" ?>
             </td>
         </tr>
             <?php
@@ -358,11 +358,11 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="sql.php?<?php echo $common_url_query; ?>&amp;db=mysql&amp;sql_query=<?php echo urlencode('SHOW PROCESSLIST'); ?>&amp;goto=main.php">
-                    <?php echo $strMySQLShowProcess; ?></a>&nbsp;
-                <?php echo PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_PROCESSLIST') . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="sql.php?<?php echo  $common_url_query ?>&amp;db=mysql&amp;sql_query=<?php echo  urlencode('SHOW PROCESSLIST') ?>&amp;goto=main.php">
+                    <?php echo  $strMySQLShowProcess ?></a>&nbsp;
+                <?php echo  PMA_showMySQLDocu('MySQL_Database_Administration', 'SHOW_PROCESSLIST') . "\n" ?>
             </td>
         </tr>
             <?php
@@ -372,11 +372,11 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="main.php?<?php echo $common_url_query; ?>&amp;mode=reload">
-                    <?php echo $strReloadMySQL; ?></a>&nbsp;
-                <?php echo PMA_showMySQLDocu('MySQL_Database_Administration', 'FLUSH') . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="main.php?<?php echo  $common_url_query ?>&amp;mode=reload">
+                    <?php echo  $strReloadMySQL ?></a>&nbsp;
+                <?php echo  PMA_showMySQLDocu('MySQL_Database_Administration', 'FLUSH') . "\n" ?>
             </td>
         </tr>
             <?php
@@ -386,11 +386,11 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="user_details.php?<?php echo $common_url_query; ?>&amp;db=mysql&amp;table=user">
-                    <?php echo $strUsers; ?></a>&nbsp;
-                <?php echo PMA_showMySQLDocu('MySQL_Database_Administration', 'Privilege_system') . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="user_details.php?<?php echo  $common_url_query ?>&amp;db=mysql&amp;table=user">
+                    <?php echo  $strUsers ?></a>&nbsp;
+                <?php echo  PMA_showMySQLDocu('MySQL_Database_Administration', 'Privilege_system') . "\n" ?>
             </td>
         </tr>
             <?php
@@ -398,10 +398,10 @@
                 echo "\n";
                 ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="db_stats.php?<?php echo $common_url_query; ?>">
-                    <?php echo $strDatabasesStats; ?></a>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="db_stats.php?<?php echo  $common_url_query ?>">
+                    <?php echo  $strDatabasesStats ?></a>
             </td>
         </tr>
                 <?php
@@ -413,10 +413,10 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="user_password.php?<?php echo $common_url_query; ?>">
-                    <?php echo ($strChangePassword); ?></a>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="user_password.php?<?php echo  $common_url_query ?>">
+                    <?php echo  ($strChangePassword) ?></a>
             </td>
         </tr>
             <?php
@@ -430,10 +430,10 @@
             echo "\n";
             ?>
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="index.php?<?php echo $common_url_query; ?>&amp;old_usr=<?php echo urlencode($PHP_AUTH_USER); ?>" target="_parent">
-                    <b><?php echo $strLogout; ?></b></a>&nbsp;<?php echo $http_logout . "\n"; ?>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="index.php?<?php echo  $common_url_query ?>&amp;old_usr=<?php echo  urlencode($PHP_AUTH_USER) ?>" target="_parent">
+                    <b><?php echo  $strLogout ?></b></a>&nbsp;<?php echo  $http_logout . "\n" ?>
             </td>
         </tr>
             <?php
@@ -456,7 +456,7 @@
 ?>
 
     <!-- phpMyAdmin related links -->
-    <td valign="top" align="<?php echo $cell_align_left; ?>">
+    <td valign="top" align="<?php echo  $cell_align_left ?>">
         <table>
         <tr>
             <th colspan="2">&nbsp;&nbsp;phpMyAdmin</th>
@@ -468,12 +468,12 @@
     ?>
         <!-- Language Selection -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
             <td nowrap="nowrap">
                 <form method="post" action="index.php" target="_parent">
-                    <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-                    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-                    Language <a href="./translators.php" target="documentation">(*)</a>:
+                    <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+                    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+                    Language <a href="./translators.php" target="documentation">(*)</a>:
                     <select name="lang" dir="ltr" onchange="this.form.submit();">
     <?php
     echo "\n";
@@ -520,12 +520,12 @@
     ?>
         <!-- Charset Selection -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
             <td nowrap="nowrap">
                 <form method="post" action="index.php" target="_parent">
-                    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-                    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-                    <?php echo $strMySQLCharset;?>:
+                    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+                    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+                    <?php echo  $strMySQLCharset ?>:
                     <select name="convcharset" dir="ltr" onchange="this.form.submit();">
     <?php
     echo "\n";
@@ -552,9 +552,9 @@
 
         <!-- Documentation -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="Documentation.php" target="documentation"><b><?php echo $strPmaDocumentation; ?></b></a>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="Documentation.php" target="documentation"><b><?php echo  $strPmaDocumentation ?></b></a>
             </td>
         </tr>
 
@@ -563,9 +563,9 @@
     ?>
         <!-- PHP Information -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="phpinfo.php" target="_new"><?php echo $strShowPHPInfo; ?></a>
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="phpinfo.php" target="_new"><?php echo  $strShowPHPInfo ?></a>
             </td>
         </tr>
     <?php
@@ -575,9 +575,9 @@
 
         <!-- phpMyAdmin related urls -->
         <tr>
-            <td valign="baseline"><img src="<?php echo $item_img; ?>" width="7" height="7" alt="item" /></td>
-            <td>
-                <a href="http://www.phpMyAdmin.net/" target="_new"><?php echo $strHomepageOfficial; ?></a><br />
+            <td valign="baseline"><img src="<?php echo  $item_img ?>" width="7" height="7" alt="item" /></td>
+            <td>
+                <a href="http://www.phpMyAdmin.net/" target="_new"><?php echo  $strHomepageOfficial ?></a><br />
                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[<a href="ChangeLog" target="_new">ChangeLog</a>]
                 &nbsp;&nbsp;&nbsp;[<a href="http://cvs.sourceforge.net/cgi-bin/viewcvs.cgi/phpmyadmin/phpMyAdmin/" target="_new">CVS</a>]
                 &nbsp;&nbsp;&nbsp;[<a href="http://sourceforge.net/mail/?group_id=23067" target="_new">Lists</a>]