--- mult_submits.inc.php (original)
+++ mult_submits.inc.php (modified)
@@ -96,7 +96,7 @@
                                 . ',';
                 }
                 if ($i == $selected_cnt-1) {
-                    $full_query = ereg_replace(',$', ';<br />', $full_query);
+                    $full_query = preg_replace(',$', ';<br />', $full_query);
                 }
                 break;
         } // end switch
@@ -106,10 +106,10 @@
     echo $strDoYouReally . '&nbsp;:<br />' . "\n";
     echo '<tt>' . $full_query . '</tt>&nbsp;?<br/>' . "\n";
     ?>
-<form action="<?php echo $action; ?>" method="post">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
+<form action="<?php echo  $action ?>" method="post">
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
     <?php
     echo "\n";
     if (strpos(' ' . $action, 'db_details') == 1) {
@@ -122,9 +122,9 @@
         echo '    <input type="hidden" name="selected[]" value="' . htmlspecialchars($selected[$i]) . '" />' . "\n";
     }
     ?>
-    <input type="hidden" name="query_type" value="<?php echo $what; ?>" />
-    <input type="submit" name="mult_btn" value="<?php echo $strYes; ?>" />
-    <input type="submit" name="mult_btn" value="<?php echo $strNo; ?>" />
+    <input type="hidden" name="query_type" value="<?php echo  $what ?>" />
+    <input type="submit" name="mult_btn" value="<?php echo  $strYes ?>" />
+    <input type="submit" name="mult_btn" value="<?php echo  $strNo ?>" />
 </form>
     <?php
     echo"\n";