--- tbl_create.php (original)
+++ tbl_create.php (modified)
@@ -94,8 +94,8 @@
     } // end for
     unset($field_cnt);
     unset($query);
-    $sql_query = ereg_replace(', $', '', $sql_query);
-    $query_cpy = ereg_replace(', $', '', $query_cpy);
+    $sql_query = preg_replace(', $', '', $sql_query);
+    $query_cpy = preg_replace(', $', '', $query_cpy);
 
     // Builds the primary keys statements
     $primary     = '';
@@ -110,7 +110,7 @@
         }
     } // end for
     unset($primary_cnt);
-    $primary = ereg_replace(', $', '', $primary);
+    $primary = preg_replace(', $', '', $primary);
     if (!empty($primary)) {
         $sql_query .= ', PRIMARY KEY (' . $primary . ')';
         $query_cpy .= ',' . "\n" . '  PRIMARY KEY (' . $primary . ')';
@@ -130,7 +130,7 @@
         }
     } // end for
     unset($index_cnt);
-    $index = ereg_replace(', $', '', $index);
+    $index = preg_replace(', $', '', $index);
     if (!empty($index)) {
         $sql_query .= ', INDEX (' . $index . ')';
         $query_cpy .= ',' . "\n" . '  INDEX (' . $index . ')';
@@ -150,7 +150,7 @@
         }
     } // end for
     unset($unique_cnt);
-    $unique = ereg_replace(', $', '', $unique);
+    $unique = preg_replace(', $', '', $unique);
     if (!empty($unique)) {
         $sql_query .= ', UNIQUE (' . $unique . ')';
         $query_cpy .= ',' . "\n" . '  UNIQUE (' . $unique . ')';
@@ -170,7 +170,7 @@
         }
     } // end for
     unset($field_fulltext);
-    $fulltext = ereg_replace(', $', '', $fulltext);
+    $fulltext = preg_replace(', $', '', $fulltext);
     if (!empty($fulltext)) {
         $sql_query .= ', FULLTEXT (' . $fulltext . ')';
         $query_cpy .= ',' . "\n" . '  FULLTEXT (' . $fulltext . ')';