--- tbl_alter.php (original)
+++ tbl_alter.php (modified)
@@ -64,7 +64,7 @@
         $query .= PMA_backquote($field_orig[$i]) . ' ' . PMA_backquote($field_name[$i]) . ' ' . $field_type[$i];
         // Some field types shouldn't have lengths
         if ($field_length[$i] != ''
-            && !eregi('^(DATE|DATETIME|TIME|TINYBLOB|TINYTEXT|BLOB|TEXT|MEDIUMBLOB|MEDIUMTEXT|LONGBLOB|LONGTEXT)$', $field_type[$i])) {
+            && !preg_match('^(DATE|DATETIME|TIME|TINYBLOB|TINYTEXT|BLOB|TEXT|MEDIUMBLOB|MEDIUMTEXT|LONGBLOB|LONGTEXT)$', $field_type[$i])) {
             $query .= '(' . $field_length[$i] . ')';
         }
         if ($field_attribute[$i] != '') {
@@ -124,7 +124,7 @@
         $local_query   = 'SHOW FIELDS FROM ' . PMA_backquote($table) . ' FROM ' . PMA_backquote($db) . " LIKE '$field'";
         $result        = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, '', $err_url);
         $fields_meta[] = PMA_mysql_fetch_array($result);
-        mysql_free_result($result);
+        mysqli_free_result($result);
     }
 
     $num_fields  = count($fields_meta);