--- tbl_properties_options.php (original)
+++ tbl_properties_options.php (modified)
@@ -59,14 +59,14 @@
     <!-- Table comments -->
     <li>
         <form method="post" action="tbl_properties_options.php">
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-            <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
-            <?php echo $strTableComments; ?>&nbsp;:&nbsp;
-            <input type="hidden" name="prev_comment" value="<?php echo urlencode($show_comment); ?>" />&nbsp;
-            <input type="text" name="comment" maxlength="60" size="30" value="<?php echo htmlspecialchars($show_comment); ?>" class="textfield" style="vertical-align: middle" onfocus="this.select()" />&nbsp;
-            <input type="submit" name="submitcomment" value="<?php echo $strGo; ?>" style="vertical-align: middle" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+            <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
+            <?php echo  $strTableComments ?>&nbsp;:&nbsp;
+            <input type="hidden" name="prev_comment" value="<?php echo  urlencode($show_comment) ?>" />&nbsp;
+            <input type="text" name="comment" maxlength="60" size="30" value="<?php echo  htmlspecialchars($show_comment) ?>" class="textfield" style="vertical-align: middle" onfocus="this.select()" />&nbsp;
+            <input type="submit" name="submitcomment" value="<?php echo  $strGo ?>" style="vertical-align: middle" />
         </form>
     </li>
 
@@ -75,7 +75,7 @@
     // modify robbat2 code - staybyte - 11. June 2001
     $query  = 'SHOW VARIABLES LIKE \'have_%\'';
     $result = PMA_mysql_query($query);
-    if ($result != FALSE && mysql_num_rows($result) > 0) {
+    if ($result != FALSE && mysqli_num_rows($result) > 0) {
         while ($tmp = PMA_mysql_fetch_array($result)) {
             if (isset($tmp['Variable_name'])) {
                 switch ($tmp['Variable_name']) {
@@ -104,16 +104,16 @@
         } // end while
     } // end if $result
 
-    mysql_free_result($result);
+    mysqli_free_result($result);
     echo "\n";
     ?>
     <li>
         <form method="post" action="tbl_properties_options.php">
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-            <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
-            <?php echo $strTableType; ?>&nbsp;:&nbsp;
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+            <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
+            <?php echo  $strTableType ?>&nbsp;:&nbsp;
             <select name="tbl_type" style="vertical-align: middle">
                 <option value="MYISAM"<?php if ($tbl_type == 'MYISAM') echo ' selected="selected"'; ?>>MyISAM</option>
                 <option value="HEAP"<?php if ($tbl_type == 'HEAP') echo ' selected="selected"'; ?>>Heap</option>
@@ -144,8 +144,8 @@
     ?>
                 <option value="MERGE"<?php if ($tbl_type == 'MRG_MYISAM') echo ' selected="selected"'; ?>>Merge</option>
             </select>&nbsp;
-            <input type="submit" name="submittype" value="<?php echo $strGo; ?>" style="vertical-align: middle" />&nbsp;
-            <?php echo PMA_showMySQLDocu('Table_types', 'Table_types') . "\n"; ?>
+            <input type="submit" name="submittype" value="<?php echo  $strGo ?>" style="vertical-align: middle" />&nbsp;
+            <?php echo  PMA_showMySQLDocu('Table_types', 'Table_types') . "\n" ?>
         </form>
     </li>
 
@@ -155,29 +155,29 @@
         <tr>
             <td valign="top">
                 <form method="post" action="tbl_properties_options.php">
-                    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-                    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-                    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-                    <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
+                    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+                    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+                    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+                    <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
 
                     <table border="0" cellspacing="0" cellpadding="0">
                     <tr>
                         <td>
                             <input type="checkbox" name="pack_keys" id="pack_keys_opt"
-                                <?php echo (isset($pack_keys) && $pack_keys == 1) ? ' checked="checked"' : ''; ?> />
+                                <?php echo  (isset($pack_keys) && $pack_keys == 1) ? ' checked="checked"' : '' ?> />
                             <label for="pack_keys_opt">pack_keys</label>&nbsp;&nbsp;
                             <br />
                             <input type="checkbox" name="checksum" id="checksum_opt"
-                                <?php echo (isset($checksum) && $checksum == 1) ? ' checked="checked"' : ''; ?> />
+                                <?php echo  (isset($checksum) && $checksum == 1) ? ' checked="checked"' : '' ?> />
                             <label for="checksum_opt">checksum</label>&nbsp;&nbsp;
                             <br />
                             <input type="checkbox" name="delay_key_write" id="delay_key_write_opt"
-                                <?php echo (isset($delay_key_write) && $delay_key_write == 1) ? ' checked="checked"' : ''; ?> />
+                                <?php echo  (isset($delay_key_write) && $delay_key_write == 1) ? ' checked="checked"' : '' ?> />
                             <label for="delay_key_write_opt">delay_key_write</label>&nbsp;&nbsp;
                             &nbsp;&nbsp;
                         </td>
                         <td>
-                            <input type="submit" name="submitoptions" value="<?php echo $strGo; ?>" />
+                            <input type="submit" name="submitoptions" value="<?php echo  $strGo ?>" />
                         </td>
                     </tr>
                     </table>