--- tbl_properties_table_info.php (original)
+++ tbl_properties_table_info.php (modified)
@@ -34,7 +34,7 @@
     $table_info_num_rows = PMA_mysql_result($table_info_result, 0, 'count');
     $show_comment        = '';
 }
-mysql_free_result($table_info_result);
+mysqli_free_result($table_info_result);
 
 
 /**
@@ -51,7 +51,7 @@
     ?>
 <!-- Table comment -->
 <p><i>
-    <?php echo htmlspecialchars($show_comment) . "\n"; ?>
+    <?php echo  htmlspecialchars($show_comment) . "\n" ?>
 </i></p>
     <?php
 } // end if