--- tbl_relation.php (original)
+++ tbl_relation.php (modified)
@@ -111,7 +111,7 @@
                    . ' AND table_name = \'' . PMA_sqlAddslashes($table) . '\''
                    . ' AND column_name = \'' . PMA_handleSlashes($key) . '\'';
         $test_rs   = PMA_query_as_cu($test_qry);
-        if ($test_rs && mysql_num_rows($test_rs) > 0) {
+        if ($test_rs && mysqli_num_rows($test_rs) > 0) {
             if (strlen($value) > 0) {
                 $upd_query = 'UPDATE ' . PMA_backquote($cfgRelation['column_comments'])
                            . ' SET comment = \'' . PMA_handleSlashes($value) . '\''
@@ -164,7 +164,7 @@
         if (($curr_table[0] != $table) && ($curr_table[0] != $cfg['Server']['relation'])) {
             $fi_query = 'SHOW KEYS FROM ' . PMA_backquote($curr_table[0]);
             $fi_rs    = PMA_mysql_query($fi_query) or PMA_mysqlDie('', $fi_query, '', $err_url_0);
-            if ($fi_rs && mysql_num_rows($fi_rs) > 0) {
+            if ($fi_rs && mysqli_num_rows($fi_rs) > 0) {
                 while ($curr_field = PMA_mysql_fetch_array($fi_rs)) {
                     if (isset($curr_field['Key_name']) && $curr_field['Key_name'] == 'PRIMARY') {
                         $field_full = $db . '.' .$curr_field['Table'] . '.' . $curr_field['Column_name'];
@@ -179,14 +179,14 @@
                 if (isset($field_full) && isset($field_v)) {
                     $selectboxall[$field_full] =  $field_v;
                 }
-            } // end if (mysql_num_rows)
+            } // end if (mysqli_num_rows)
         // Mike Beck - 24.07.02: i've been asked to add all keys of the
         // current table (see bug report #574851)
         }
         else if ($curr_table[0] == $table) {
             $fi_query = 'SHOW KEYS FROM ' . PMA_backquote($curr_table[0]);
             $fi_rs    = PMA_mysql_query($fi_query) or PMA_mysqlDie('', $fi_query, '', $err_url_0);
-            if ($fi_rs && mysql_num_rows($fi_rs) > 0) {
+            if ($fi_rs && mysqli_num_rows($fi_rs) > 0) {
                 while ($curr_field = PMA_mysql_fetch_array($fi_rs)) {
                     $field_full = $db . '.' . $curr_field['Table'] . '.' . $curr_field['Column_name'];
                     $field_v    = $curr_field['Table'] . '->' . $curr_field['Column_name'];
@@ -194,7 +194,7 @@
                         $selectboxall[$field_full] =  $field_v;
                     }
                 } // end while
-            } // end if (mysql_num_rows)
+            } // end if (mysqli_num_rows)
         }
     } // end while over tables
 
@@ -206,7 +206,7 @@
 $col_query = 'SHOW COLUMNS FROM ' . PMA_backquote($table);
 $col_rs    = PMA_mysql_query($col_query) or PMA_mysqlDie('', $col_query, '', $err_url_0);
 
-if ($col_rs && mysql_num_rows($col_rs) > 0) {
+if ($col_rs && mysqli_num_rows($col_rs) > 0) {
     while ($row = PMA_mysql_fetch_array($col_rs)) {
         $save_row[] = $row;
     }
@@ -214,25 +214,25 @@
 
     ?>
 <form method="post" action="tbl_relation.php">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-    <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+    <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
     <input type="hidden" name="submit_rel" value="true" />
 
     <table>
     <tr>
-        <th colspan="2" align="center"><b><?php echo $strLinksTo; ?></b></th>
+        <th colspan="2" align="center"><b><?php echo  $strLinksTo ?></b></th>
     </tr>
     <?php
     for ($i = 0; $i < $saved_row_cnt; $i++) {
         echo "\n";
         ?>
     <tr>
-        <th><?php echo $save_row[$i]['Field']; ?></th>
+        <th><?php echo  $save_row[$i]['Field'] ?></th>
         <td>
-            <input type="hidden" name="src_field" value="<?php echo $save_row[$i]['Field']; ?>" />
-            <select name="destination[<?php echo htmlspecialchars($save_row[$i]['Field']); ?>]">
+            <input type="hidden" name="src_field" value="<?php echo  $save_row[$i]['Field'] ?>" />
+            <select name="destination[<?php echo  htmlspecialchars($save_row[$i]['Field']) ?>]">
         <?php
         echo "\n";
         reset($selectboxall);
@@ -275,7 +275,7 @@
     ?>
     <tr>
         <td colspan="2" align="center">
-            <input type="submit" value="<?php echo $strGo; ?>" />
+            <input type="submit" value="<?php echo  $strGo ?>" />
         </td>
     </tr>
     </table>
@@ -289,17 +289,17 @@
         echo "\n";
         ?>
 <form method="post" action="tbl_relation.php">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-    <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+    <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
     <input type="hidden" name="submit_show" value="true" />
 
-    <p><?php echo $strChangeDisplay; ?></p>
+    <p><?php echo  $strChangeDisplay ?></p>
     <select name="display_field" onchange="this.form.submit();">
         <?php
         echo "\n";
-        mysql_data_seek($col_rs, 0);
+        mysqli_data_seek($col_rs, 0);
         while ($row = @PMA_mysql_fetch_array($col_rs)) {
             echo '        <option value="' . htmlspecialchars($row['Field']) . '"';
             if (isset($disp) && $row['Field'] == $disp) {
@@ -315,7 +315,7 @@
     //-->
     </script>
     <noscript>
-        <input type="submit" value="<?php echo $strGo; ?>" />
+        <input type="submit" value="<?php echo  $strGo ?>" />
     </noscript>
 </form>
         <?php
@@ -326,15 +326,15 @@
         echo "\n";
         ?>
 <form method="post" action="tbl_relation.php">
-    <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-    <input type="hidden" name="server" value="<?php echo $server; ?>" />
-    <input type="hidden" name="db" value="<?php echo htmlspecialchars($db); ?>" />
-    <input type="hidden" name="table" value="<?php echo htmlspecialchars($table); ?>" />
+    <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+    <input type="hidden" name="server" value="<?php echo  $server ?>" />
+    <input type="hidden" name="db" value="<?php echo  htmlspecialchars($db) ?>" />
+    <input type="hidden" name="table" value="<?php echo  htmlspecialchars($table) ?>" />
     <input type="hidden" name="submit_comm" value="true" />
 
     <table>
     <tr>
-        <th colspan="2" align="center"><b><?php echo $strComments; ?></b></th>
+        <th colspan="2" align="center"><b><?php echo  $strComments ?></b></th>
     </tr>
         <?php
         for ($i = 0; $i < $saved_row_cnt; $i++) {
@@ -342,9 +342,9 @@
             echo "\n";
             ?>
     <tr>
-        <th><?php echo $field; ?></th>
+        <th><?php echo  $field ?></th>
         <td>
-            <input type="text" name="comment[<?php echo $field; ?>]" value="<?php echo (isset($comments[$field]) ?  htmlspecialchars($comments[$field]) : ''); ?>" />
+            <input type="text" name="comment[<?php echo  $field ?>]" value="<?php echo  (isset($comments[$field]) ?  htmlspecialchars($comments[$field]) : '') ?>" />
         </td>
     </tr>
             <?php
@@ -354,7 +354,7 @@
         ?>
     <tr>
         <td colspan="2" align="center">
-            <input type="submit" value="<?php echo $strGo; ?>" />
+            <input type="submit" value="<?php echo  $strGo ?>" />
         </td>
     </tr>
     </table>