--- tbl_replace.php (original)
+++ tbl_replace.php (modified)
@@ -50,7 +50,7 @@
           . '&sql_query=' . urlencode($sql_query);
 } else if (!empty($goto)) {
     // Security checkings
-    $is_gotofile     = ereg_replace('^([^?]+).*$', '\\1', $goto);
+    $is_gotofile     = preg_replace('^([^?]+).*$', '\\1', $goto);
     if (!@file_exists('./' . $is_gotofile)) {
         $goto        = (empty($table)) ? 'db_details.php' : 'tbl_properties.php';
         $is_gotofile = TRUE;
@@ -160,7 +160,7 @@
             if (empty($funcs[$encoded_key])) {
                 $valuelist .= PMA_backquote($key) . ' = ' . $val . ', ';
             } else if ($val == '\'\''
-                       && (ereg('^(NOW|CURDATE|CURTIME|UNIX_TIMESTAMP|RAND|USER|LAST_INSERT_ID)$', $funcs[$encoded_key]))) {
+                       && (preg_match('^(NOW|CURDATE|CURTIME|UNIX_TIMESTAMP|RAND|USER|LAST_INSERT_ID)$', $funcs[$encoded_key]))) {
                 $valuelist .= PMA_backquote($key) . ' = ' . $funcs[$encoded_key] . '(), ';
             } else {
                 $valuelist .= PMA_backquote($key) . ' = ' . $funcs[$encoded_key] . "($val), ";
@@ -169,7 +169,7 @@
     } // end while
 
     // Builds the sql update query
-    $valuelist    = ereg_replace(', $', '', $valuelist);
+    $valuelist    = preg_replace(', $', '', $valuelist);
     if (!empty($valuelist)) {
         $query    = 'UPDATE ' . PMA_backquote($table) . ' SET ' . $valuelist . ' WHERE' . $primary_key
                   . ((PMA_MYSQL_INT_VERSION >= 32300) ? ' LIMIT 1' : '');
@@ -181,7 +181,7 @@
         if ($is_gotofile) {
             $js_to_run = 'functions.js';
             include('./header.inc.php');
-            include('./' . ereg_replace('\.\.*', '.', $goto));
+            include('./' . preg_replace('\.\.*', '.', $goto));
         } else {
             header('Location: ' . $cfg['PmaAbsoluteUri'] . $goto . '&message=' . urlencode($message));
         }
@@ -262,8 +262,8 @@
         if (empty($funcs[$encoded_key])) {
             $valuelist .= $val . ', ';
         } else if (($val == '\'\''
-                   && ereg('^(UNIX_TIMESTAMP|RAND|LAST_INSERT_ID)$', $funcs[$encoded_key]))
-                   || ereg('^(NOW|CURDATE|CURTIME|USER)$', $funcs[$encoded_key])) {
+                   && preg_match('^(UNIX_TIMESTAMP|RAND|LAST_INSERT_ID)$', $funcs[$encoded_key]))
+                   || preg_match('^(NOW|CURDATE|CURTIME|USER)$', $funcs[$encoded_key])) {
             $valuelist .= $funcs[$encoded_key] . '(), ';
         } else {
             $valuelist .= $funcs[$encoded_key] . '(' . $val . '), ';
@@ -271,8 +271,8 @@
     } // end while
 
     // Builds the sql insert query
-    $fieldlist = ereg_replace(', $', '', $fieldlist);
-    $valuelist = ereg_replace(', $', '', $valuelist);
+    $fieldlist = preg_replace(', $', '', $fieldlist);
+    $valuelist = preg_replace(', $', '', $valuelist);
     $query     = 'INSERT INTO ' . PMA_backquote($table) . ' (' . $fieldlist . ') VALUES (' . $valuelist . ')';
     $message   = $strInsertedRows . '&nbsp;';
 } // end row insertion
@@ -291,8 +291,8 @@
     include('./header.inc.php');
     PMA_mysqlDie($error, '', '', $err_url);
 } else {
-    if (@mysql_affected_rows()) {
-        $message .= @mysql_affected_rows();
+    if (@mysqli_affected_rows()) {
+        $message .= @mysqli_affected_rows();
     } else {
         $message = $strModifications;
     }
@@ -302,7 +302,7 @@
         }
         $js_to_run = 'functions.js';
         include('./header.inc.php');
-        include('./' . ereg_replace('\.\.*', '.', $goto));
+        include('./' . preg_replace('\.\.*', '.', $goto));
     } else {
         $add_query = (strpos(' ' . $goto, 'tbl_change') ? '&disp_query=' . urlencode($sql_query) : '');
         header('Location: ' . $cfg['PmaAbsoluteUri'] . $goto . '&message=' . urlencode($message) . $add_query);