--- tbl_printview.php (original)
+++ tbl_printview.php (modified)
@@ -97,7 +97,7 @@
         $show_comment = '';
     } // end display comments
     if ($result) {
-        mysql_free_result($result);
+        mysqli_free_result($result);
     }
 
 
@@ -140,7 +140,7 @@
 
     } // end while
     if ($result) {
-        mysql_free_result($result);
+        mysqli_free_result($result);
     }
 
 
@@ -149,7 +149,7 @@
      */
     $local_query = 'SHOW FIELDS FROM ' . PMA_backquote($table);
     $result      = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, '', $err_url);
-    $fields_cnt  = mysql_num_rows($result);
+    $fields_cnt  = mysqli_num_rows($result);
 
     // Check if we can use Relations (Mike Beck)
     if (!empty($cfgRelation['relation'])) {
@@ -183,12 +183,12 @@
 <!-- TABLE INFORMATIONS -->
 <table width="100%" bordercolorlight="black" border="border" style="border-collapse: collapse; background-color: white">
 <tr>
-    <th width="50"><?php echo $strField; ?></th>
-    <th width="50"><?php echo $strType; ?></th>
-    <!--<th width="50"><?php echo $strAttr; ?></th>-->
-    <th width="50"><?php echo $strNull; ?></th>
-    <th width="50"><?php echo $strDefault; ?></th>
-    <!--<th width="50"><?php echo $strExtra; ?></th>-->
+    <th width="50"><?php echo  $strField ?></th>
+    <th width="50"><?php echo  $strType ?></th>
+    <!--<th width="50"><?php echo  $strAttr ?></th>-->
+    <th width="50"><?php echo  $strNull ?></th>
+    <th width="50"><?php echo  $strDefault ?></th>
+    <!--<th width="50"><?php echo  $strExtra ?></th>-->
     <?php
     echo "\n";
     if ($have_rel) {
@@ -209,23 +209,23 @@
         $type             = $row['Type'];
         // reformat mysql query output - staybyte - 9. June 2001
         // loic1: set or enum types: slashes single quotes inside options
-        if (eregi('^(set|enum)\((.+)\)$', $type, $tmp)) {
-            $tmp[2]       = substr(ereg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
+        if (preg_match('^(set|enum)\((.+)\)$', $type, $tmp)) {
+            $tmp[2]       = substr(preg_replace('([^,])\'\'', '\\1\\\'', ',' . $tmp[2]), 1);
             $type         = $tmp[1] . '(' . str_replace(',', ', ', $tmp[2]) . ')';
             $type_nowrap  = '';
         } else {
             $type_nowrap  = ' nowrap="nowrap"';
         }
-        $type             = eregi_replace('BINARY', '', $type);
-        $type             = eregi_replace('ZEROFILL', '', $type);
-        $type             = eregi_replace('UNSIGNED', '', $type);
+        $type             = preg_replace('BINARY', '', $type);
+        $type             = preg_replace('ZEROFILL', '', $type);
+        $type             = preg_replace('UNSIGNED', '', $type);
         if (empty($type)) {
             $type         = '&nbsp;';
         }
 
-        $binary           = eregi('BINARY', $row['Type'], $test);
-        $unsigned         = eregi('UNSIGNED', $row['Type'], $test);
-        $zerofill         = eregi('ZEROFILL', $row['Type'], $test);
+        $binary           = preg_match('BINARY', $row['Type'], $test);
+        $unsigned         = preg_match('UNSIGNED', $row['Type'], $test);
+        $zerofill         = preg_match('ZEROFILL', $row['Type'], $test);
         $strAttribute     = '&nbsp;';
         if ($binary) {
             $strAttribute = 'BINARY';
@@ -256,11 +256,11 @@
     }
     ?>
     </td>
-    <td width="50" class="print"<?php echo $type_nowrap; ?>><?php echo $type; ?><bdo dir="ltr"></bdo></td>
-    <!--<td width="50" bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $strAttribute; ?></td>-->
-    <td width="50" class="print"><?php echo (($row['Null'] == '') ? $strNo : $strYes); ?>&nbsp;</td>
+    <td width="50" class="print"<?php echo  $type_nowrap ?>><?php echo  $type ?><bdo dir="ltr"></bdo></td>
+    <!--<td width="50" bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $strAttribute ?></td>-->
+    <td width="50" class="print"><?php echo  (($row['Null'] == '') ? $strNo : $strYes) ?>&nbsp;</td>
     <td width="50" class="print" nowrap="nowrap"><?php if (isset($row['Default'])) echo $row['Default']; ?>&nbsp;</td>
-    <!--<td width="50" bgcolor="<?php echo $bgcolor; ?>" nowrap="nowrap"><?php echo $row['Extra']; ?>&nbsp;</td>-->
+    <!--<td width="50" bgcolor="<?php echo  $bgcolor ?>" nowrap="nowrap"><?php echo  $row['Extra'] ?>&nbsp;</td>-->
     <?php
     echo "\n";
     if ($have_rel) {
@@ -282,7 +282,7 @@
 </tr>
         <?php
     } // end while
-    mysql_free_result($result);
+    mysqli_free_result($result);
 
     echo "\n";
     ?>
@@ -302,13 +302,13 @@
 <br /><br />
 
 <!-- Indexes -->
-&nbsp;<big><?php echo $strIndexes . '&nbsp;:'; ?></big>
+&nbsp;<big><?php echo  $strIndexes . '&nbsp;:' ?></big>
 <table bordercolorlight="black" border="border" style="border-collapse: collapse; background-color: white">
     <tr>
-        <th><?php echo $strKeyname; ?></th>
-        <th><?php echo $strType; ?></th>
-        <th><?php echo $strCardinality; ?></th>
-        <th colspan="2"><?php echo $strField; ?></th>
+        <th><?php echo  $strKeyname ?></th>
+        <th><?php echo  $strType ?></th>
+        <th><?php echo  $strCardinality ?></th>
+        <th colspan="2"><?php echo  $strField ?></th>
     </tr>
         <?php
         echo "\n";
@@ -372,7 +372,7 @@
      */
     if ($cfg['ShowStats']) {
         $nonisam     = FALSE;
-        if (isset($showtable['Type']) && !eregi('ISAM|HEAP', $showtable['Type'])) {
+        if (isset($showtable['Type']) && !preg_match('ISAM|HEAP', $showtable['Type'])) {
             $nonisam = TRUE;
         }
         if (PMA_MYSQL_INT_VERSION >= 32303 && $nonisam == FALSE) {
@@ -407,25 +407,25 @@
 
     <!-- Space usage -->
     <td class="print" valign="top">
-        &nbsp;<big><?php echo $strSpaceUsage . '&nbsp;:'; ?></big>
+        &nbsp;<big><?php echo  $strSpaceUsage . '&nbsp;:' ?></big>
         <table width="100%" bordercolorlight="black" border="border" style="border-collapse: collapse; background-color: white">
         <tr>
-            <th><?php echo $strType; ?></th>
-            <th colspan="2" align="center"><?php echo $strUsage; ?></th>
-        </tr>
-        <tr>
-            <td class="print" style="padding-right: 10px"><?php echo $strData; ?></td>
-            <td align="right" class="print" nowrap="nowrap"><?php echo $data_size; ?></td>
-            <td class="print"><?php echo $data_unit; ?></td>
+            <th><?php echo  $strType ?></th>
+            <th colspan="2" align="center"><?php echo  $strUsage ?></th>
+        </tr>
+        <tr>
+            <td class="print" style="padding-right: 10px"><?php echo  $strData ?></td>
+            <td align="right" class="print" nowrap="nowrap"><?php echo  $data_size ?></td>
+            <td class="print"><?php echo  $data_unit ?></td>
         </tr>
             <?php
             if (isset($index_size)) {
                 echo "\n";
                 ?>
         <tr>
-            <td class="print" style="padding-right: 10px"><?php echo $strIndex; ?></td>
-            <td align="right" class="print" nowrap="nowrap"><?php echo $index_size; ?></td>
-            <td class="print"><?php echo $index_unit; ?></td>
+            <td class="print" style="padding-right: 10px"><?php echo  $strIndex ?></td>
+            <td align="right" class="print" nowrap="nowrap"><?php echo  $index_size ?></td>
+            <td class="print"><?php echo  $index_unit ?></td>
         </tr>
                 <?php
             }
@@ -433,14 +433,14 @@
                 echo "\n";
                 ?>
         <tr style="color: #bb0000">
-            <td class="print" style="padding-right: 10px"><?php echo $strOverhead; ?></td>
-            <td align="right" class="print" nowrap="nowrap"><?php echo $free_size; ?></td>
-            <td class="print"><?php echo $free_unit; ?></td>
-        </tr>
-        <tr>
-            <td class="print" style="padding-right: 10px"><?php echo $strEffective; ?></td>
-            <td align="right" class="print" nowrap="nowrap"><?php echo $effect_size; ?></td>
-            <td class="print"><?php echo $effect_unit; ?></td>
+            <td class="print" style="padding-right: 10px"><?php echo  $strOverhead ?></td>
+            <td align="right" class="print" nowrap="nowrap"><?php echo  $free_size ?></td>
+            <td class="print"><?php echo  $free_unit ?></td>
+        </tr>
+        <tr>
+            <td class="print" style="padding-right: 10px"><?php echo  $strEffective ?></td>
+            <td align="right" class="print" nowrap="nowrap"><?php echo  $effect_size ?></td>
+            <td class="print"><?php echo  $effect_unit ?></td>
         </tr>
                 <?php
             }
@@ -448,9 +448,9 @@
                 echo "\n";
                 ?>
         <tr>
-            <td class="print" style="padding-right: 10px"><?php echo $strTotalUC; ?></td>
-            <td align="right" class="print" nowrap="nowrap"><?php echo $tot_size; ?></td>
-            <td class="print"><?php echo $tot_unit; ?></td>
+            <td class="print" style="padding-right: 10px"><?php echo  $strTotalUC ?></td>
+            <td align="right" class="print" nowrap="nowrap"><?php echo  $tot_size ?></td>
+            <td class="print"><?php echo  $tot_unit ?></td>
         </tr>
                 <?php
             }
@@ -463,11 +463,11 @@
 
     <!-- Rows Statistic -->
     <td valign="top">
-        &nbsp;<big><?php echo $strRowsStatistic . '&nbsp;:'; ?></big>
+        &nbsp;<big><?php echo  $strRowsStatistic . '&nbsp;:' ?></big>
         <table width=100% bordercolorlight="black" border="border" style="border-collapse: collapse; background-color: white">
         <tr>
-            <th><?php echo $strStatement; ?></th>
-            <th align="center"><?php echo $strValue; ?></th>
+            <th><?php echo  $strStatement ?></th>
+            <th align="center"><?php echo  $strValue ?></th>
         </tr>
             <?php
             $i = 0;
@@ -476,8 +476,8 @@
                 echo "\n";
                 ?>
         <tr>
-            <td class="print"><?php echo ucfirst($strFormat); ?></td>
-            <td align="<?php echo $cell_align_left; ?>" class="print" nowrap="nowrap">
+            <td class="print"><?php echo  ucfirst($strFormat) ?></td>
+            <td align="<?php echo  $cell_align_left ?>" class="print" nowrap="nowrap">
                 <?php
                 echo '                ';
                 if ($showtable['Row_format'] == 'Fixed') {
@@ -498,9 +498,9 @@
                 echo "\n";
             ?>
         <tr>
-            <td class="print"><?php echo ucfirst($strRows); ?></td>
+            <td class="print"><?php echo  ucfirst($strRows) ?></td>
             <td align="right" class="print" nowrap="nowrap">
-                <?php echo number_format($showtable['Rows'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+                <?php echo  number_format($showtable['Rows'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
                 <?php
@@ -510,9 +510,9 @@
                 echo "\n";
                 ?>
         <tr>
-            <td class="print"><?php echo ucfirst($strRowLength); ?>&nbsp;&oslash;</td>
+            <td class="print"><?php echo  ucfirst($strRowLength) ?>&nbsp;&oslash;</td>
             <td class="print" nowrap="nowrap">
-                <?php echo number_format($showtable['Avg_row_length'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+                <?php echo  number_format($showtable['Avg_row_length'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
                 <?php
@@ -522,9 +522,9 @@
                 echo "\n";
                 ?>
         <tr>
-            <td class="print"><?php echo ucfirst($strRowSize); ?>&nbsp;&oslash;</td>
+            <td class="print"><?php echo  ucfirst($strRowSize) ?>&nbsp;&oslash;</td>
             <td align="right" class="print" nowrap="nowrap">
-                <?php echo $avg_size . ' ' . $avg_unit . "\n"; ?>
+                <?php echo  $avg_size . ' ' . $avg_unit . "\n" ?>
             </td>
         </tr>
                 <?php
@@ -534,9 +534,9 @@
                 echo "\n";
                 ?>
         <tr>
-            <td class="print"><?php echo ucfirst($strNext); ?>&nbsp;Autoindex</td>
+            <td class="print"><?php echo  ucfirst($strNext) ?>&nbsp;Autoindex</td>
             <td align="right" class="print" nowrap="nowrap">
-                <?php echo number_format($showtable['Auto_increment'], 0, $number_decimal_separator, $number_thousands_separator) . "\n"; ?>
+                <?php echo  number_format($showtable['Auto_increment'], 0, $number_decimal_separator, $number_thousands_separator) . "\n" ?>
             </td>
         </tr>
                 <?php