--- config.auth.lib.php (original)
+++ config.auth.lib.php (modified)
@@ -97,23 +97,23 @@
         ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $GLOBALS['available_languages'][$GLOBALS['lang']][2]; ?>" lang="<?php echo $GLOBALS['available_languages'][$GLOBALS['lang']][2]; ?>" dir="<?php echo $GLOBALS['text_dir']; ?>">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo  $GLOBALS['available_languages'][$GLOBALS['lang']][2] ?>" lang="<?php echo  $GLOBALS['available_languages'][$GLOBALS['lang']][2] ?>" dir="<?php echo  $GLOBALS['text_dir'] ?>">
 
 <head>
-<title><?php echo $GLOBALS['strAccessDenied']; ?></title>
-<meta http-equiv="Content-Type" content="text/html; charset=<?php echo $GLOBALS['charset']; ?>" />
+<title><?php echo  $GLOBALS['strAccessDenied'] ?></title>
+<meta http-equiv="Content-Type" content="text/html; charset=<?php echo  $GLOBALS['charset'] ?>" />
 <style type="text/css">
 <!--
-body     {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000}
-h1       {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold}
+body     {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000}
+h1       {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold}
 //-->
 </style>
 </head>
 
-<body bgcolor="<?php echo $cfg['RightBgColor']; ?>">
+<body bgcolor="<?php echo  $cfg['RightBgColor'] ?>">
 <br /><br />
 <center>
-    <h1><?php echo sprintf($GLOBALS['strWelcome'], ' phpMyAdmin ' . PMA_VERSION); ?></h1>
+    <h1><?php echo  sprintf($GLOBALS['strWelcome'], ' phpMyAdmin ' . PMA_VERSION) ?></h1>
 </center>
 <br />
         <?php