--- upload.php (original)
+++ upload.php (modified)
@@ -1,3 +1,4 @@
+<?php
 <?php
 $TopDir="../../ImgFolder";
 
@@ -26,10 +27,10 @@
 
 if(is_uploaded_file($MlangMultyFile)) {
 
-	$full_filename = split("\.", $MlangMultyFile_name);
+	$full_filename = explode("\.", $MlangMultyFile_name);
 	$file_extention = $full_filename[sizeof($MlangMultyFile_name)];
 
-	if (eregi($file_extention, "html|php3|phtml|inc|asp")) {
+	if (preg_match($file_extention, "html|php3|phtml|inc|asp")) {
 		$msg = "\\nphp / asp 관련파일은 직접 업로드 하실 수 없습니다.\\n\\n파일의 확장자를 변경하여 올려 주세요.\\n";
 		echo ("<script language=javascript>
                      window.alert('$msg');