--- charset_conversion.lib.php (original)
+++ charset_conversion.lib.php (modified)
@@ -144,7 +144,7 @@
     /**
      * Converts encoding according to current settings.
      *
-     * @param   mixed    what to convert (string or array of strings or object returned by mysql_fetch_field)
+     * @param   mixed    what to convert (string or array of strings or object returned by mysqli_fetch_field)
      *
      * @return  string   converted string or array of strings
      *
@@ -195,7 +195,7 @@
             }
         }
         else if (is_object($what)) {
-            // isn't it object returned from mysql_fetch_field ?
+            // isn't it object returned from mysqli_fetch_field ?
             if (@is_string($what->name)) {
                 $what->name = PMA_convert_display_charset($what->name);
             }