--- db_table_exists.lib.php (original)
+++ db_table_exists.lib.php (modified)
@@ -23,11 +23,11 @@
         $is_table = @PMA_mysql_query('SHOW TABLES LIKE \'' . PMA_sqlAddslashes($table, TRUE) . '\'');
     }
     if (empty($table)
-        || !($is_table && @mysql_numrows($is_table))) {
+        || !($is_table && @mysqli_numrows($is_table))) {
         header('Location: ' . $cfg['PmaAbsoluteUri'] . 'db_details.php?lang=' . $lang . '&convcharset=' . $convcharset . '&server=' . $server . '&db=' . urlencode($db) . (isset($message) ? '&message=' . urlencode($message) : '') . '&reload=1');
         exit();
     } else if (isset($is_table)) {
-        mysql_free_result($is_table);
+        mysqli_free_result($is_table);
     }
 } // end if (ensures table exists)
 ?>