--- display_tbl.lib.php (original)
+++ display_tbl.lib.php (modified)
@@ -95,7 +95,7 @@
             // 2.2 Statement is a "SHOW..."
             else if ($GLOBALS['is_show']) {
                 // 2.2.1 TODO : defines edit/delete links depending on show statement
-                $tmp = eregi('^SHOW[[:space:]]+(VARIABLES|(FULL[[:space:]]+)?PROCESSLIST|STATUS|TABLE|GRANTS|CREATE|LOGS)', $GLOBALS['sql_query'], $which);
+                $tmp = preg_match('^SHOW[[:space:]]+(VARIABLES|(FULL[[:space:]]+)?PROCESSLIST|STATUS|TABLE|GRANTS|CREATE|LOGS)', $GLOBALS['sql_query'], $which);
                 if (strpos(' ' . strtoupper($which[1]), 'PROCESSLIST') > 0) {
                     $do_display['edit_lnk'] = 'nn'; // no edit link
                     $do_display['del_lnk']  = 'kp'; // "kill process" type edit link
@@ -124,7 +124,7 @@
                                 || $do_display['sort_lnk'] != '0'
                                 || $do_display['ins_row'] != '0');
                     // 2.3.1 Displays text cut/expand button?
-                    if ($do_display['text_btn'] == '0' && eregi('BLOB', $fields_meta[$i]->type)) {
+                    if ($do_display['text_btn'] == '0' && preg_match('BLOB', $fields_meta[$i]->type)) {
                         $do_display['text_btn'] = (string) '1';
                         if (!$is_link) {
                             break;
@@ -159,7 +159,7 @@
             $local_query = 'SELECT COUNT(*) AS total FROM ' . PMA_backquote($db) . '.' . PMA_backquote($table);
             $result      = PMA_mysql_query($local_query) or PMA_mysqlDie('', $local_query, '', $err_url);
             $the_total   = PMA_mysql_result($result, 0, 'total');
-            mysql_free_result($result);
+            mysqli_free_result($result);
         }
 
         // 4. If navigation bar or sorting fields names urls should be
@@ -238,36 +238,36 @@
             ?>
     <td>
         <form action="sql.php" method="post">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
             <input type="hidden" name="pos" value="0" />
-            <input type="hidden" name="session_max_rows" value="<?php echo $session_max_rows; ?>" />
-            <input type="hidden" name="disp_direction" value="<?php echo $disp_direction; ?>" />
-            <input type="hidden" name="repeat_cells" value="<?php echo $repeat_cells; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $caption1; ?>"<?php echo $title1; ?> />
+            <input type="hidden" name="session_max_rows" value="<?php echo  $session_max_rows ?>" />
+            <input type="hidden" name="disp_direction" value="<?php echo  $disp_direction ?>" />
+            <input type="hidden" name="repeat_cells" value="<?php echo  $repeat_cells ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $caption1 ?>"<?php echo  $title1 ?> />
         </form>
     </td>
     <td>
         <form action="sql.php" method="post">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
-            <input type="hidden" name="pos" value="<?php echo $pos_prev; ?>" />
-            <input type="hidden" name="session_max_rows" value="<?php echo $session_max_rows; ?>" />
-            <input type="hidden" name="disp_direction" value="<?php echo $disp_direction; ?>" />
-            <input type="hidden" name="repeat_cells" value="<?php echo $repeat_cells; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $caption2; ?>"<?php echo $title2; ?> />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
+            <input type="hidden" name="pos" value="<?php echo  $pos_prev ?>" />
+            <input type="hidden" name="session_max_rows" value="<?php echo  $session_max_rows ?>" />
+            <input type="hidden" name="disp_direction" value="<?php echo  $disp_direction ?>" />
+            <input type="hidden" name="repeat_cells" value="<?php echo  $repeat_cells ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $caption2 ?>"<?php echo  $title2 ?> />
         </form>
     </td>
             <?php
@@ -279,19 +279,19 @@
     </td>
     <td align="center">
         <form action="sql.php" method="post"
-            onsubmit="return (checkFormElementInRange(this, 'session_max_rows', 1) && checkFormElementInRange(this, 'pos', 0, <?php echo $unlim_num_rows - 1; ?>))">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $GLOBALS['strShow']; ?>&nbsp;:" />
-            <input type="text" name="session_max_rows" size="3" value="<?php echo (($session_max_rows != 'all') ? $session_max_rows : $GLOBALS['cfg']['MaxRows']); ?>" class="textfield" onfocus="this.select()" />
-            <?php echo $GLOBALS['strRowsFrom'] . "\n"; ?>
-            <input type="text" name="pos" size="6" value="<?php echo (($pos_next >= $unlim_num_rows) ? 0 : $pos_next); ?>" class="textfield" onfocus="this.select()" />
+            onsubmit="return (checkFormElementInRange(this, 'session_max_rows', 1) && checkFormElementInRange(this, 'pos', 0, <?php echo  $unlim_num_rows - 1 ?>))">
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $GLOBALS['strShow'] ?>&nbsp;:" />
+            <input type="text" name="session_max_rows" size="3" value="<?php echo  (($session_max_rows != 'all') ? $session_max_rows : $GLOBALS['cfg']['MaxRows']) ?>" class="textfield" onfocus="this.select()" />
+            <?php echo  $GLOBALS['strRowsFrom'] . "\n" ?>
+            <input type="text" name="pos" size="6" value="<?php echo  (($pos_next >= $unlim_num_rows) ? 0 : $pos_next) ?>" class="textfield" onfocus="this.select()" />
             <br />
         <?php
         // Display mode (horizontal/vertical and repeat headers)
@@ -329,37 +329,37 @@
             ?>
     <td>
         <form action="sql.php" method="post">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
-            <input type="hidden" name="pos" value="<?php echo $pos_next; ?>" />
-            <input type="hidden" name="session_max_rows" value="<?php echo $session_max_rows; ?>" />
-            <input type="hidden" name="disp_direction" value="<?php echo $disp_direction; ?>" />
-            <input type="hidden" name="repeat_cells" value="<?php echo $repeat_cells; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $caption3; ?>"<?php echo $title3; ?> />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
+            <input type="hidden" name="pos" value="<?php echo  $pos_next ?>" />
+            <input type="hidden" name="session_max_rows" value="<?php echo  $session_max_rows ?>" />
+            <input type="hidden" name="disp_direction" value="<?php echo  $disp_direction ?>" />
+            <input type="hidden" name="repeat_cells" value="<?php echo  $repeat_cells ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $caption3 ?>"<?php echo  $title3 ?> />
         </form>
     </td>
     <td>
         <form action="sql.php" method="post"
-            onsubmit="return <?php echo (($pos + $session_max_rows < $unlim_num_rows && $num_rows >= $session_max_rows) ? 'true' : 'false'); ?>">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
-            <input type="hidden" name="pos" value="<?php echo $unlim_num_rows - $session_max_rows; ?>" />
-            <input type="hidden" name="session_max_rows" value="<?php echo $session_max_rows; ?>" />
-            <input type="hidden" name="disp_direction" value="<?php echo $disp_direction; ?>" />
-            <input type="hidden" name="repeat_cells" value="<?php echo $repeat_cells; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $caption4; ?>"<?php echo $title4; ?> />
+            onsubmit="return <?php echo  (($pos + $session_max_rows < $unlim_num_rows && $num_rows >= $session_max_rows) ? 'true' : 'false') ?>">
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
+            <input type="hidden" name="pos" value="<?php echo  $unlim_num_rows - $session_max_rows ?>" />
+            <input type="hidden" name="session_max_rows" value="<?php echo  $session_max_rows ?>" />
+            <input type="hidden" name="disp_direction" value="<?php echo  $disp_direction ?>" />
+            <input type="hidden" name="repeat_cells" value="<?php echo  $repeat_cells ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $caption4 ?>"<?php echo  $title4 ?> />
         </form>
     </td>
             <?php
@@ -374,19 +374,19 @@
     </td>
     <td>
         <form action="sql.php" method="post">
-            <input type="hidden" name="lang" value="<?php echo $lang; ?>" />
-            <input type="hidden" name="convcharset" value="<?php echo $convcharset; ?>" />
-            <input type="hidden" name="server" value="<?php echo $server; ?>" />
-            <input type="hidden" name="db" value="<?php echo $db; ?>" />
-            <input type="hidden" name="table" value="<?php echo $table; ?>" />
-            <input type="hidden" name="sql_query" value="<?php echo $encoded_query; ?>" />
+            <input type="hidden" name="lang" value="<?php echo  $lang ?>" />
+            <input type="hidden" name="convcharset" value="<?php echo  $convcharset ?>" />
+            <input type="hidden" name="server" value="<?php echo  $server ?>" />
+            <input type="hidden" name="db" value="<?php echo  $db ?>" />
+            <input type="hidden" name="table" value="<?php echo  $table ?>" />
+            <input type="hidden" name="sql_query" value="<?php echo  $encoded_query ?>" />
             <input type="hidden" name="pos" value="0" />
             <input type="hidden" name="session_max_rows" value="all" />
-            <input type="hidden" name="disp_direction" value="<?php echo $disp_direction; ?>" />
-            <input type="hidden" name="repeat_cells" value="<?php echo $repeat_cells; ?>" />
-            <input type="hidden" name="goto" value="<?php echo $goto; ?>" />
-            <input type="hidden" name="dontlimitchars" value="<?php echo $dontlimitchars; ?>" />
-            <input type="submit" name="navig" value="<?php echo $GLOBALS['strShowAll']; ?>" />
+            <input type="hidden" name="disp_direction" value="<?php echo  $disp_direction ?>" />
+            <input type="hidden" name="repeat_cells" value="<?php echo  $repeat_cells ?>" />
+            <input type="hidden" name="goto" value="<?php echo  $goto ?>" />
+            <input type="hidden" name="dontlimitchars" value="<?php echo  $dontlimitchars ?>" />
+            <input type="submit" name="navig" value="<?php echo  $GLOBALS['strShowAll'] ?>" />
         </form>
     </td>
             <?php
@@ -481,9 +481,9 @@
             $vertical_display['emptypre'] = ($is_display['edit_lnk'] != 'nn' && $is_display['del_lnk'] != 'nn') ? 2 : 1;
             if ($disp_direction == 'horizontal') {
                 ?>
-    <td colspan="<?php echo $fields_cnt; ?>" align="center">
-        <a href="<?php echo $text_url; ?>">
-            <img src="./images/<?php echo (($dontlimitchars) ? 'partialtext' : 'fulltext'); ?>.png" border="0" width="50" height="20" alt="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" title="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" /></a>
+    <td colspan="<?php echo  $fields_cnt ?>" align="center">
+        <a href="<?php echo  $text_url ?>">
+            <img src="./images/<?php echo  (($dontlimitchars) ? 'partialtext' : 'fulltext') ?>.png" border="0" width="50" height="20" alt="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" title="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" /></a>
     </td>
 </tr>
 
@@ -494,9 +494,9 @@
                 echo "\n";
                 ?>
 <tr>
-    <td colspan="<?php echo $num_rows + floor($num_rows/$repeat_cells) + 1; ?>" align="center">
-        <a href="<?php echo $text_url; ?>">
-            <img src="./images/<?php echo (($dontlimitchars) ? 'partialtext' : 'fulltext'); ?>.png" border="0" width="50" height="20" alt="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" title="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" /></a>
+    <td colspan="<?php echo  $num_rows + floor($num_rows/$repeat_cells) + 1 ?>" align="center">
+        <a href="<?php echo  $text_url ?>">
+            <img src="./images/<?php echo  (($dontlimitchars) ? 'partialtext' : 'fulltext') ?>.png" border="0" width="50" height="20" alt="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" title="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" /></a>
     </td>
 </tr>
                 <?php
@@ -510,9 +510,9 @@
             if ($disp_direction == 'horizontal') {
                 echo "\n";
                 ?>
-    <td<?php echo $colspan; ?> align="center">
-        <a href="<?php echo $text_url; ?>">
-            <img src="./images/<?php echo (($dontlimitchars) ? 'partialtext' : 'fulltext'); ?>.png" border="0" width="50" height="20" alt="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" title="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" /></a>
+    <td<?php echo  $colspan ?> align="center">
+        <a href="<?php echo  $text_url ?>">
+            <img src="./images/<?php echo  (($dontlimitchars) ? 'partialtext' : 'fulltext') ?>.png" border="0" width="50" height="20" alt="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" title="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" /></a>
     </td>
                 <?php
             } // end horizontal mode
@@ -531,7 +531,7 @@
             if ($disp_direction == 'horizontal') {
                 echo "\n";
                 ?>
-    <td<?php echo $colspan; ?>></td>
+    <td<?php echo  $colspan ?>></td>
                 <?php
                 echo "\n";
             } // end horizontal mode
@@ -544,7 +544,7 @@
         // 2.0 If sorting links should be used, checks if the query is a "JOIN"
         //     statement (see 2.1.3)
         if ($is_display['sort_lnk'] == '1') {
-            $is_join = eregi('(.*)[[:space:]]+FROM[[:space:]]+.*[[:space:]]+JOIN', $sql_query, $select_stt);
+            $is_join = preg_match('(.*)[[:space:]]+FROM[[:space:]]+.*[[:space:]]+JOIN', $sql_query, $select_stt);
         } else {
             $is_join = FALSE;
         }
@@ -554,22 +554,22 @@
             if ($is_display['sort_lnk'] == '1') {
                 // Defines the url used to append/modify a sorting order
                 // 2.1.1 Checks if an hard coded 'order by' clause exists
-                if (eregi('(.*)([[:space:]]ORDER[[:space:]]*BY[[:space:]](.*))', $sql_query, $regs1)) {
-                    if (eregi('((.*)([[:space:]]ASC|[[:space:]]DESC)([[:space:]]|$))(.*)', $regs1[2], $regs2)) {
+                if (preg_match('(.*)([[:space:]]ORDER[[:space:]]*BY[[:space:]](.*))', $sql_query, $regs1)) {
+                    if (preg_match('((.*)([[:space:]]ASC|[[:space:]]DESC)([[:space:]]|$))(.*)', $regs1[2], $regs2)) {
                         $unsorted_sql_query = trim($regs1[1] . ' ' . $regs2[5]);
                         $sql_order          = trim($regs2[1]);
-                        eregi('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
+                        preg_match('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
                         $sort_expression = trim($after_order[2]);
                     }
-                    else if (eregi('((.*))[[:space:]]+(LIMIT (.*)|PROCEDURE (.*)|FOR UPDATE|LOCK IN SHARE MODE)', $regs1[2], $regs3)) {
+                    else if (preg_match('((.*))[[:space:]]+(LIMIT (.*)|PROCEDURE (.*)|FOR UPDATE|LOCK IN SHARE MODE)', $regs1[2], $regs3)) {
                         $unsorted_sql_query = trim($regs1[1] . ' ' . $regs3[3]);
                         $sql_order          = trim($regs3[1]) . ' ASC';
-                        eregi('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
+                        preg_match('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
                         $sort_expression = trim($after_order[2]);
                     } else {
                         $unsorted_sql_query = trim($regs1[1]);
                         $sql_order          = trim($regs1[2]) . ' ASC';
-                        eregi('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
+                        preg_match('(ORDER[[:space:]]*BY[[:space:]]*)(.*)([[:space:]]*ASC|[[:space:]]*DESC)',$sql_order,$after_order);
                         $sort_expression = trim($after_order[2]);
                     }
                 } else {
@@ -587,8 +587,8 @@
 
                     // field name may be preceded by a space, or any number
                     // of characters followed by a dot (tablename.fieldname)
-//                    $is_in_sort = eregi('([[:space:]]|(.*\.))(`?)' . $pattern . '(`?)[ ,$]', $sql_order);
-                    // instead of using eregi(), now do a direct comparison
+//                    $is_in_sort = preg_match('([[:space:]]|(.*\.))(`?)' . $pattern . '(`?)[ ,$]', $sql_order);
+                    // instead of using preg_match(), now do a direct comparison
                     // for the sort expression (avoids problems with queries
                     // like "SELECT id, count(id)..." and clicking to sort
                     // on id or on count(id) )
@@ -598,7 +598,7 @@
                 //       for a query with a "JOIN" statement and if the column
                 //       isn't aliased)
                 if ($is_join
-                    && !eregi('([^[:space:],]|`[^`]`)[[:space:]]+(as[[:space:]]+)?' . $fields_meta[$i]->name, $select_stt[1], $parts)) {
+                    && !preg_match('([^[:space:],]|`[^`]`)[[:space:]]+(as[[:space:]]+)?' . $fields_meta[$i]->name, $select_stt[1], $parts)) {
                     $sort_tbl = PMA_backquote($fields_meta[$i]->table) . '.';
                 } else {
                     $sort_tbl = '';
@@ -616,20 +616,20 @@
                     // loic1: patch #455484 ("Smart" order)
                     $cfg['Order']  = strtoupper($GLOBALS['cfg']['Order']);
                     if ($cfg['Order'] == 'SMART') {
-                        $cfg['Order'] = (eregi('time|date', $fields_meta[$i]->type)) ? 'DESC' : 'ASC';
+                        $cfg['Order'] = (preg_match('time|date', $fields_meta[$i]->type)) ? 'DESC' : 'ASC';
                     }
                     $sort_order .= $cfg['Order'];
                     $order_img   = '';
                 }
-                else if (eregi('[[:space:]]ASC$', $sql_order)) {
+                else if (preg_match('[[:space:]]ASC$', $sql_order)) {
                     $sort_order .= ' DESC';
                     $order_img   = '&nbsp;<img src="./images/asc_order.gif" border="0" width="7" height="7" alt="'. $GLOBALS['strAscending'] . '" title="'. $GLOBALS['strAscending'] . '" />';
                 }
-                else if (eregi('[[:space:]]DESC$', $sql_order)) {
+                else if (preg_match('[[:space:]]DESC$', $sql_order)) {
                     $sort_order .= ' ASC';
                     $order_img   = '&nbsp;<img src="./images/desc_order.gif" border="0" width="7" height="7" alt="'. $GLOBALS['strDescending'] . '" title="'. $GLOBALS['strDescending'] . '" />';
                 }
-                if (eregi('(.*)([[:space:]](LIMIT (.*)|PROCEDURE (.*)|FOR UPDATE|LOCK IN SHARE MODE))', $unsorted_sql_query, $regs3)) {
+                if (preg_match('(.*)([[:space:]](LIMIT (.*)|PROCEDURE (.*)|FOR UPDATE|LOCK IN SHARE MODE))', $unsorted_sql_query, $regs3)) {
                     $sorted_sql_query = $regs3[1] . $sort_order . $regs3[2];
                 } else {
                     $sorted_sql_query = $unsorted_sql_query . $sort_order;
@@ -651,8 +651,8 @@
                     echo "\n";
                     ?>
     <th>
-        <a href="sql.php?<?php echo $url_query; ?>">
-            <?php echo htmlspecialchars($fields_meta[$i]->name); ?></a><?php echo $order_img . "\n"; ?>
+        <a href="sql.php?<?php echo  $url_query ?>">
+            <?php echo  htmlspecialchars($fields_meta[$i]->name) ?></a><?php echo  $order_img . "\n" ?>
     </th>
                     <?php
                 }
@@ -668,7 +668,7 @@
                     echo "\n";
                     ?>
     <th>
-        <?php echo htmlspecialchars($fields_meta[$i]->name) . "\n"; ?>
+        <?php echo  htmlspecialchars($fields_meta[$i]->name) . "\n" ?>
     </th>
                     <?php
                 }
@@ -687,9 +687,9 @@
             if ($disp_direction == 'horizontal') {
                 echo "\n";
                 ?>
-    <td<?php echo $colspan; ?> align="center">
-        <a href="<?php echo $text_url; ?>">
-            <img src="./images/<?php echo (($dontlimitchars) ? 'partialtext' : 'fulltext'); ?>.png" border="0" width="50" height="20" alt="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" title="<?php echo (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']); ?>" /></a>
+    <td<?php echo  $colspan ?> align="center">
+        <a href="<?php echo  $text_url ?>">
+            <img src="./images/<?php echo  (($dontlimitchars) ? 'partialtext' : 'fulltext') ?>.png" border="0" width="50" height="20" alt="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" title="<?php echo  (($dontlimitchars) ? $GLOBALS['strPartialText'] : $GLOBALS['strFullText']) ?>" /></a>
     </td>
                 <?php
             } // end horizontal mode
@@ -710,7 +710,7 @@
             if ($disp_direction == 'horizontal') {
                 echo "\n";
                 ?>
-    <td<?php echo $colspan; ?>></td>
+    <td<?php echo  $colspan ?>></td>
                 <?php
             } // end horizontal mode
             else {
@@ -876,7 +876,7 @@
                     }
                 } // end if
                 ?>
-<tr<?php echo $on_mouse; ?>>
+<tr<?php echo  $on_mouse ?>>
                 <?php
                 echo "\n";
             }
@@ -945,7 +945,7 @@
                     } else {
                         $uva_condition = $uva_nonprimary_condition;
                     }
-                    $uva_condition     = urlencode(ereg_replace('[[:space:]]?AND$', '', $uva_condition));
+                    $uva_condition     = urlencode(preg_replace('[[:space:]]?AND$', '', $uva_condition));
                 } // end if (1.1)
 
                 // 1.2 Defines the urls for the modify/delete link(s)
@@ -1044,7 +1044,7 @@
                 //      (works only if function_exists('is_null')
                 // PS:   why not always work with the number ($i), since
                 //       the default second parameter of
-                //       mysql_fetch_array() is MYSQL_BOTH, so we always get
+                //       mysqli_fetch_array() is MYSQL_BOTH, so we always get
                 //       associative and numeric indices?
 
                     //if (!isset($row[$meta->name])
@@ -1073,7 +1073,7 @@
                                              . ' WHERE ' . PMA_backquote($map[$meta->name][1])
                                              . ' = ' . $row[$pointer];
                                 $dispresult  = PMA_mysql_query($dispsql);
-                                if ($dispresult && mysql_num_rows($dispresult) > 0) {
+                                if ($dispresult && mysqli_num_rows($dispresult) > 0) {
                                     $dispval = PMA_mysql_result($dispresult, 0);
                                 }
                                 else {
@@ -1102,13 +1102,13 @@
 
                 //  b l o b
 
-                } else if ($GLOBALS['cfg']['ShowBlob'] == FALSE && eregi('BLOB', $meta->type)) {
+                } else if ($GLOBALS['cfg']['ShowBlob'] == FALSE && preg_match('BLOB', $meta->type)) {
                     // loic1 : PMA_mysql_fetch_fields returns BLOB in place of
                     // TEXT fields type, however TEXT fields must be displayed
                     // even if $cfg['ShowBlob'] is false -> get the true type
                     // of the fields.
                     $field_flags = PMA_mysql_field_flags($dt_result, $i);
-                    if (eregi('BINARY', $field_flags)) {
+                    if (preg_match('BINARY', $field_flags)) {
                         $vertical_display['data'][$row_no][$i]     = '    <td align="center" valign="top" bgcolor="' . $bgcolor . '">[BLOB]</td>' . "\n";
                     } else {
                         //if (!isset($row[$meta->name])
@@ -1123,7 +1123,7 @@
                             // characters for tabulations and <cr>/<lf>
                             $row[$pointer]     = htmlspecialchars($row[$pointer]);
                             $row[$pointer]     = str_replace("\011", ' &nbsp;&nbsp;&nbsp;', str_replace('  ', ' &nbsp;', $row[$pointer]));
-                            $row[$pointer]     = ereg_replace("((\015\012)|(\015)|(\012))", '<br />', $row[$pointer]);
+                            $row[$pointer]     = preg_replace("((\015\012)|(\015)|(\012))", '<br />', $row[$pointer]);
                             $vertical_display['data'][$row_no][$i] = '    <td valign="top" bgcolor="' . $bgcolor . '">' . $row[$pointer] . '</td>' . "\n";
                         } else {
                             $vertical_display['data'][$row_no][$i] = '    <td valign="top" bgcolor="' . $bgcolor . '">&nbsp;</td>' . "\n";
@@ -1139,14 +1139,14 @@
                         $relation_id = $row[$pointer];
 
                         // loic1: Cut text/blob fields even if $cfg['ShowBlob'] is true
-                        if (eregi('BLOB', $meta->type)) {
+                        if (preg_match('BLOB', $meta->type)) {
                             if (strlen($row[$pointer]) > $GLOBALS['cfg']['LimitChars'] && ($dontlimitchars != 1)) {
                                 $row[$pointer] = substr($row[$pointer], 0, $GLOBALS['cfg']['LimitChars']) . '...';
                             }
                         }
                         // loic1: displays special characters from binaries
                         $field_flags = PMA_mysql_field_flags($dt_result, $i);
-                        if (eregi('BINARY', $field_flags)) {
+                        if (preg_match('BINARY', $field_flags)) {
                             $row[$pointer]     = str_replace("\x00", '\0', $row[$pointer]);
                             $row[$pointer]     = str_replace("\x08", '\b', $row[$pointer]);
                             $row[$pointer]     = str_replace("\x0a", '\n', $row[$pointer]);
@@ -1159,10 +1159,10 @@
                         else {
                             $row[$pointer]     = htmlspecialchars($row[$pointer]);
                             $row[$pointer]     = str_replace("\011", ' &nbsp;&nbsp;&nbsp;', str_replace('  ', ' &nbsp;', $row[$pointer]));
-                            $row[$pointer]     = ereg_replace("((\015\012)|(\015)|(\012))", '<br />', $row[$pointer]);
+                            $row[$pointer]     = preg_replace("((\015\012)|(\015)|(\012))", '<br />', $row[$pointer]);
                         }
                         // loic1: do not wrap if date field type
-                        $nowrap = (eregi('DATE|TIME', $meta->type) ? ' nowrap="nowrap"' : '');
+                        $nowrap = (preg_match('DATE|TIME', $meta->type) ? ' nowrap="nowrap"' : '');
                         $vertical_display['data'][$row_no][$i]     = '    <td valign="top" bgcolor="' . $bgcolor . '"' . $nowrap . '>';
 
                         reset($analyzed_sql[0]['select_expr']);
@@ -1184,7 +1184,7 @@
                                              . ' WHERE ' . PMA_backquote($map[$meta->name][1])
                                              . ' = \'' . PMA_sqlAddslashes($row[$pointer]) . '\'';
                                 $dispresult  = @PMA_mysql_query($dispsql);
-                                if ($dispresult && mysql_num_rows($dispresult) > 0) {
+                                if ($dispresult && mysqli_num_rows($dispresult) > 0) {
                                     $dispval = PMA_mysql_result($dispresult, 0);
                                 }
                                 else {
@@ -1507,9 +1507,9 @@
         if ($cfgRelation['relwork']) {
             // find tables
             //$pattern = '`?[[:space:]]+(((ON|on)[[:space:]]+[^,]+)?,|((NATURAL|natural)[[:space:]]+)?(INNER|inner|LEFT|left|RIGHT|right)([[:space:]]+(OUTER|outer))?[[:space:]]+(JOIN|join))[[:space:]]*`?';
-            //$target  = eregi_replace('^.*[[:space:]]+FROM[[:space:]]+`?|`?[[:space:]]*(ON[[:space:]]+[^,]+)?(WHERE[[:space:]]+.*)?$', '', $sql_query);
-            //$target = eregi_replace('`?[[:space:]]ORDER BY[[:space:]](.*)','',$target);
-            //$tabs    = '(\'' . join('\',\'', split($pattern, $target)) . '\')';
+            //$target  = preg_replace('^.*[[:space:]]+FROM[[:space:]]+`?|`?[[:space:]]*(ON[[:space:]]+[^,]+)?(WHERE[[:space:]]+.*)?$', '', $sql_query);
+            //$target = preg_replace('`?[[:space:]]ORDER BY[[:space:]](.*)','',$target);
+            //$tabs    = '(\'' . join('\',\'', explode($pattern, $target)) . '\')';
             $target=array();
             reset($analyzed_sql[0]['table_ref']);
             while (list ($table_ref_position, $table_ref) = each ($analyzed_sql[0]['table_ref'])) {