--- relation.lib.php (original)
+++ relation.lib.php (modified)
@@ -34,7 +34,7 @@
             PMA_mysql_select_db($cfgRelation['db'], $dbh);
             $result = @PMA_mysql_query($sql, $dbh);
             if (!$result && $show_error == TRUE) {
-                PMA_mysqlDie(mysql_error($dbh), $sql, '', $err_url_0);
+                PMA_mysqlDie(mysqli_error($dbh), $sql, '', $err_url_0);
             }
             PMA_mysql_select_db($db, $dbh);
         } else {
@@ -147,7 +147,7 @@
             $cfgRelation['allworks'] = TRUE;
         }
         if ($tab_rs) {
-            mysql_free_result($tab_rs);
+            mysqli_free_result($tab_rs);
         } else {
             $cfg['Server']['pmadb'] = FALSE;
         }