--- decode_bug.php (original)
+++ decode_bug.php (modified)
@@ -46,7 +46,7 @@
 <br />
 
 <form method="post" action="./decode_bug.php">
-    <input type="hidden" name="bar" value="<?php echo rand(); ?>" />
+    <input type="hidden" name="bar" value="<?php echo  rand() ?>" />
     Encoded bug report:<br />
     <textarea name="bug_encoded" cols="72" rows="10"></textarea>
     <br /><br />
@@ -86,7 +86,7 @@
         $bug_encoded = stripslashes($bug_encoded);
     }
 
-    $bug_encoded     = ereg_replace('[[:space:]]', '', $bug_encoded);
+    $bug_encoded     = preg_replace('[[:space:]]', '', $bug_encoded);
     $bug_decoded     = base64_decode($bug_encoded);
     if (substr($bug_encoded, 0, 2) == 'eN') {
         if (function_exists('gzuncompress')) {
@@ -106,3 +106,4 @@
 </body>
 
 </html>
+?>