--- data_edit.php (original)
+++ data_edit.php (modified)
@@ -6,7 +6,7 @@
    $connect = dbconn();
 
   $query = "select * from shop_d1"; 
-  $result = mysqli_query($query, $connect); 
+  $result = mysqli_query($connect, $query); 
   if(!$result) die(mysqli_error()); 
   $data = mysqli_fetch_array($result); 
 ?> 
@@ -45,7 +45,7 @@
 <br><br>
 <?php
   $query = "select * from shop_d2"; 
-  $result = mysqli_query($query, $connect); 
+  $result = mysqli_query($connect, $query); 
   if(!$result) die(mysqli_error()); 
   $data = mysqli_fetch_array($result); 
 ?>   
@@ -83,7 +83,7 @@
 <br><br>
 <?php
   $query = "select * from shop_d3"; 
-  $result = mysqli_query($query, $connect); 
+  $result = mysqli_query($connect, $query); 
   if(!$result) die(mysqli_error()); 
   $data = mysqli_fetch_array($result); 
 ?>
@@ -117,7 +117,7 @@
      <?php
 
 $query = "select * from shop_d4"; 
-$result = mysqli_query($query, $connect); 
+$result = mysqli_query($connect, $query); 
 if(!$result) die(mysqli_error()); 
 $data = mysqli_fetch_array($result); 
 ?>