--- lib02.php (original)
+++ lib02.php (modified)
@@ -1,3 +1,4 @@
+<?php
 <?php
    $id = "duson1830";
    $pw = "du1830";
@@ -21,9 +22,9 @@
   }
   
   function dbconn(){
-      $connect = mysql_connect("localhost","duson1830","du1830");
-      mysql_select_db("duson1830",$connect);
-      mysql_query("SET NAMES utf-8");
+      $connect = mysqli_connect("localhost","duson1830","du1830");
+      mysqli_select_db($connect, "duson1830");
+      mysqli_query("SET NAMES utf-8");
       return $connect;
     }  
   
@@ -31,3 +32,4 @@
 <style>
 td,input,li{font-size:9pt}
 </style>
+?>