--- order_list.php (original)
+++ order_list.php (modified)
@@ -12,7 +12,7 @@
 
   // 전체 페이지 구하기 
   $query = "select count(*) from shop_list "; 
-  $result = mysqli_query($query, $connect); 
+  $result = mysqli_query($connect, $query); 
   $data = mysqli_fetch_array($result); 
   $ap = $data[0]; 
   echo "<li> 총 게시물수 : $ap  "; 
@@ -30,7 +30,7 @@
   $query = "select * from shop_list order by no desc"; /**/
   $query .= " limit $s, $pagenum "; 
  echo $query; 
-  $result = mysqli_query($query, $connect);
+  $result = mysqli_query($connect, $query);
   ?> 
 <table width=100% border=1> 
   <tr> 