--- post_all_list.php (original)
+++ post_all_list.php (modified)
@@ -12,7 +12,7 @@
 
   // 전체 페이지 구하기 
   $query = "select count(*) from MlangOrder_PrintAuto"; 
-  $result = mysqli_query($query, $connect); 
+  $result = mysqli_query($connect, $query); 
   $data = mysqli_fetch_array($result); 
   $total = $data[0]; 
 
@@ -32,7 +32,7 @@
   //$query = "select * from MlangOrder_PrintAuto where (zip1 like '% $zip1 %') or (zip2 like '%-%') order by no desc";   
   $query .= " limit $s, $pagenum "; 
   echo $query; 
-  $result = mysqli_query($query, $connect);
+  $result = mysqli_query($connect, $query);
   ?> 
 <table width=100% border=1> 
   <tr bgcolor="#99CCFF"> 