--- malay-utf-8.inc.php (original)
+++ malay-utf-8.inc.php (modified)
@@ -298,7 +298,7 @@
 $strPdfNoTables = 'Tiada Jadual';
 $strPhp = 'Cipta Kod PHP';
 $strPmaDocumentation = 'Dokumentasi phpMyAdmin';
-$strPmaUriError = '<tt>$cfg[PmaAbsoluteUri]</tt> MESTI disetkan di dalam fail konfigurasi.';
+$strPmaUriError = '<tt>$cfg['PmaAbsoluteUri']</tt> MESTI disetkan di dalam fail konfigurasi.';
 $strPos1 = 'Mula';
 $strPrevious = 'Terdahulu';
 $strPrimary = 'Utama';