--- header_printview.inc.php (original)
+++ header_printview.inc.php (modified)
@@ -44,31 +44,31 @@
 ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $available_languages[$lang][2]; ?>" lang="<?php echo $available_languages[$lang][2]; ?>" dir="<?php echo $text_dir; ?>">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo  $available_languages[$lang][2] ?>" lang="<?php echo  $available_languages[$lang][2] ?>" dir="<?php echo  $text_dir ?>">
 
 <head>
-<title><?php echo $strSQLResult; ?> - phpMyAdmin <?php echo PMA_VERSION ?></title>
-<meta http-equiv="Content-Type" content="text/html; charset=<?php echo $charset; ?>" />
+<title><?php echo  $strSQLResult ?> - phpMyAdmin <?php echo  PMA_VERSION ?></title>
+<meta http-equiv="Content-Type" content="text/html; charset=<?php echo  $charset ?>" />
 <style type="text/css">
 <!--
-body  {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000; background-color: #ffffff}
-h1    {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_bigger; ?>; font-weight: bold}
+body  {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000; background-color: #ffffff}
+h1    {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_bigger ?>; font-weight: bold}
 table {border-width:1px; border-color:#000000; border-style:solid; border-collapse:collapse; border-spacing:0}
-th    {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; font-weight: bold; color: #000000; background-color: #ffffff; border-width:1px; border-color:#000000; border-style:solid; padding:2px}
-td    {font-family: <?php echo $right_font_family; ?>; font-size: <?php echo $font_size; ?>; color: #000000; background-color: #ffffff; border-width:1px; border-color:#000000; border-style:solid; padding:2px}
+th    {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; font-weight: bold; color: #000000; background-color: #ffffff; border-width:1px; border-color:#000000; border-style:solid; padding:2px}
+td    {font-family: <?php echo  $right_font_family ?>; font-size: <?php echo  $font_size ?>; color: #000000; background-color: #ffffff; border-width:1px; border-color:#000000; border-style:solid; padding:2px}
 //-->
 </style>
 </head>
 
 
 <body bgcolor="#ffffff">
-<h1><?php echo $strSQLResult; ?></h1>
+<h1><?php echo  $strSQLResult ?></h1>
 <p>
-    <b><?php echo $strHost; ?>:</b> <?php echo $cfg['Server']['host'] . ((!empty($cfg['Server']['port'])) ? ':' . $cfg['Server']['port'] : ''); ?><br />
-    <b><?php echo $strDatabase; ?>:</b> <?php echo htmlspecialchars($db); ?><br />
-    <b><?php echo $strGenTime; ?>:</b> <?php echo PMA_localisedDate(); ?><br />
-    <b><?php echo $strGenBy; ?>:</b> phpMyAdmin <?php echo PMA_VERSION; ?><br />
-    <b><?php echo $strSQLQuery; ?>:</b> <?php echo htmlspecialchars($full_sql_query); ?>;
+    <b><?php echo  $strHost ?>:</b> <?php echo  $cfg['Server']['host'] . ((!empty($cfg['Server']['port'])) ? ':' . $cfg['Server']['port'] : '') ?><br />
+    <b><?php echo  $strDatabase ?>:</b> <?php echo  htmlspecialchars($db) ?><br />
+    <b><?php echo  $strGenTime ?>:</b> <?php echo  PMA_localisedDate() ?><br />
+    <b><?php echo  $strGenBy ?>:</b> phpMyAdmin <?php echo  PMA_VERSION ?><br />
+    <b><?php echo  $strSQLQuery ?>:</b> <?php echo  htmlspecialchars($full_sql_query) ?>;
 </p>
 
 