--- defines.lib.php (original)
+++ defines.lib.php (modified)
@@ -15,12 +15,12 @@
 if (!defined('PMA_MYSQL_INT_VERSION') && isset($userlink)) {
     if (!empty($server)) {
         $result = PMA_mysql_query('SELECT VERSION() AS version');
-        if ($result != FALSE && @mysql_num_rows($result) > 0) {
+        if ($result != FALSE && @mysqli_num_rows($result) > 0) {
             $row   = PMA_mysql_fetch_array($result);
             $match = explode('.', $row['version']);
         } else {
             $result = @PMA_mysql_query('SHOW VARIABLES LIKE \'version\'');
-            if ($result != FALSE && @mysql_num_rows($result) > 0){
+            if ($result != FALSE && @mysqli_num_rows($result) > 0){
                 $row   = PMA_mysql_fetch_row($result);
                 $match = explode('.', $row[1]);
             }
@@ -77,19 +77,19 @@
     }
 
     // 2. browser and version
-    if (ereg('Opera(/| )([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
+    if (preg_match('Opera(/| )([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
         define('PMA_USR_BROWSER_VER', $log_version[2]);
         define('PMA_USR_BROWSER_AGENT', 'OPERA');
-    } else if (ereg('MSIE ([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
+    } else if (preg_match('MSIE ([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
         define('PMA_USR_BROWSER_VER', $log_version[1]);
         define('PMA_USR_BROWSER_AGENT', 'IE');
-    } else if (ereg('OmniWeb/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
+    } else if (preg_match('OmniWeb/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
         define('PMA_USR_BROWSER_VER', $log_version[1]);
         define('PMA_USR_BROWSER_AGENT', 'OMNIWEB');
-    } else if (ereg('Mozilla/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
+    } else if (preg_match('Mozilla/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
         define('PMA_USR_BROWSER_VER', $log_version[1]);
         define('PMA_USR_BROWSER_AGENT', 'MOZILLA');
-    } else if (ereg('Konqueror/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
+    } else if (preg_match('Konqueror/([0-9].[0-9]{1,2})', $HTTP_USER_AGENT, $log_version)) {
         define('PMA_USR_BROWSER_VER', $log_version[1]);
         define('PMA_USR_BROWSER_AGENT', 'KONQUEROR');
     } else {