--- get_foreign.lib.php (original)
+++ get_foreign.lib.php (modified)
@@ -30,7 +30,7 @@
         $count_query  = 'SELECT COUNT(*) AS total FROM ' . PMA_backquote($foreign_db) . '.' . PMA_backquote($foreign_table);
         $count_result = PMA_mysql_query($count_query) or PMA_mysqlDie('', $count_query, '', $err_url);
         $the_total    = PMA_mysql_result($count_result, 0, 'total');
-        mysql_free_result($count_result);
+        mysqli_free_result($count_result);
 
         if ($the_total < 200) {
             // foreign_display can be FALSE if no display field defined: