--- select_lang.lib.php (original)
+++ select_lang.lib.php (modified)
@@ -164,8 +164,8 @@
             while (list($key, $value) = each($available_languages)) {
                 // $envType =  1 for the 'HTTP_ACCEPT_LANGUAGE' environment variable,
                 //             2 for the 'HTTP_USER_AGENT' one
-                if (($envType == 1 && eregi('^(' . $value[0] . ')(;q=[0-9]\\.[0-9])?$', $str))
-                    || ($envType == 2 && eregi('(\(|\[|;[[:space:]])(' . $value[0] . ')(;|\]|\))', $str))) {
+                if (($envType == 1 && preg_match('^(' . $value[0] . ')(;q=[0-9]\\.[0-9])?$', $str))
+                    || ($envType == 2 && preg_match('(\(|\[|;[[:space:]])(' . $value[0] . ')(;|\]|\))', $str))) {
                     $lang     = $key;
                     break;
                 }