--- index.php (original)
+++ index.php (modified)
@@ -4,9 +4,9 @@
 
 $Mlang_query="select * from MlangHomePage_Movic";
 
-$query= mysql_query("$Mlang_query",$db);
-$recordsu= mysql_num_rows($query);
-$total = mysql_affected_rows();
+$query= mysqli_query("$Mlang_query",$db);
+$recordsu= mysqli_num_rows($query);
+$total = mysqli_affected_rows($db);
 
 $listcut= 30;  //한 페이지당 보여줄 목록 게시물수. 
 if(!$offset) $offset=0; 
@@ -22,18 +22,18 @@
 </tr>
 ");
 
-$result= mysql_query("$Mlang_query order by new='yes' desc, no desc limit $offset,$listcut",$db);
-$rows=mysql_num_rows($result);
+$result= mysqli_query("$Mlang_query order by new='yes' desc, no desc limit $offset,$listcut",$db);
+$rows=mysqli_num_rows($result);
 if($rows){
 
 $i=1+$offset;
-while($row= mysql_fetch_array($result)) 
+while($row= mysqli_fetch_array($result)) 
 { 
 ?>
 
 <tr>
    <td height=30>&nbsp;<?php echo $i?>&nbsp;</td>
-   <td colspan=2>&nbsp;<a href='#' onClick="javascript:window.open('/PHPClass/Movic/Window.php?PCode=<?php echo $row['no']?>', 'HomePageMovicWindow3','width=607,height=451,top=0,left=0,menubar=no,resizable=yes,statusbar=no,scrollbars=yes,toolbar=no');"><?php echo $row[title]?></a>&nbsp;</td>
+   <td colspan=2>&nbsp;<a href='#' onClick="javascript:window.open('/PHPClass/Movic/Window.php?PCode=<?php echo $row['no']?>', 'HomePageMovicWindow3','width=607,height=451,top=0,left=0,menubar=no,resizable=yes,statusbar=no,scrollbars=yes,toolbar=no');"><?php echo $row['title']?></a>&nbsp;</td>
 </tr>
 <tr>
 <td background='/img/line_24-2.gif' colspan=3 height=1 bgcolor='#FFFFFF'></td>
@@ -92,7 +92,7 @@
 
 }
 
-mysql_close($db); 
+mysqli_close($db); 
 ?> 
 
 </p>
@@ -101,3 +101,4 @@
 <p align=center>
 <?php echo $WebSoftCopyright?>
 </p>
+?>